#
# CDDL HEADER START
#
# The contents of this file are subject to the terms of the
# Common Development and Distribution License (the "License").  
# You may not use this file except in compliance with the License.
#
# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
# or http://www.opensolaris.org/os/licensing.
# See the License for the specific language governing permissions
# and limitations under the License.
#
# When distributing Covered Code, include this CDDL HEADER in each
# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
# If applicable, add the following below this CDDL HEADER, with the
# fields enclosed by brackets "[]" replaced with your own identifying
# information: Portions Copyright [yyyy] [name of copyright owner]
#
# CDDL HEADER END
#
# Copyright 2009 Sun Microsystems, Inc.  All rights reserved.
# Use is subject to license terms.
#
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2009-04-17 16:15+0900\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: ../client.py:117
msgid ""
"Usage:\n"
"        pkg [options] command [cmd_options] [operands]\n"
"\n"
"Basic subcommands:\n"
"        pkg install [-nvq] [--no-refresh] [--no-index] package...\n"
"        pkg uninstall [-nrvq] [--no-index] package...\n"
"        pkg list [-Hafsuv] [--no-refresh] [package...]\n"
"        pkg image-update [-fnvq] [--be-name name] [--no-refresh] [--no-"
"index]\n"
"        pkg refresh [--full] [publisher ...]\n"
"        pkg version\n"
"\n"
"Advanced subcommands:\n"
"        pkg info [-lr] [--license] [pkg_fmri_pattern ...]\n"
"        pkg search [-alprI] [-s server] query\n"
"        pkg verify [-Hqv] [pkg_fmri_pattern ...]\n"
"        pkg fix [pkg_fmri_pattern ...]\n"
"        pkg contents [-Hmr] [-o attribute ...] [-s sort_key]\n"
"            [-t action_type ... ] [pkg_fmri_pattern ...]\n"
"        pkg image-create [-fFPUz] [--force] [--full|--partial|--user] [--"
"zone]\n"
"            [-k ssl_key] [-c ssl_cert] [--no-refresh] \n"
"            [--variant <variant_spec>=<instance>]\n"
"            (-p|--publisher) name=uri dir\n"
"\n"
"        pkg set-property propname propvalue\n"
"        pkg unset-property propname ...\n"
"        pkg property [-H] [propname ...]\n"
"\n"
"        pkg set-publisher [-Ped] [-k ssl_key] [-c ssl_cert]\n"
"            [-O origin_uri] [-m mirror_to_add | --add-mirror=mirror_to_add]\n"
"            [-M mirror_to_remove | --remove-mirror=mirror_to_remove]\n"
"            [--enable] [--disable] [--no-refresh] [--reset-uuid] publisher\n"
"        pkg unset-publisher publisher ...\n"
"        pkg publisher [-HPa] [publisher ...]\n"
"        pkg history [-Hl]\n"
"        pkg purge-history\n"
"        pkg rebuild-index\n"
"\n"
"Options:\n"
"        -R dir\n"
"        --help or -?\n"
"\n"
"Environment:\n"
"        PKG_IMAGE"
msgstr ""
"Использование:\n"
"        pkg [параметры] команда [параметры команды] [операнды]\n"
"\n"
"Базовые субкоманды:\n"
"        pkg install [-nvq] [--no-refresh] [--no-index] пакет...\n"
"        pkg uninstall [-nrvq] [--no-index] пакет...\n"
"        pkg list [-Hafsuv] [--no-refresh] [пакет...]\n"
"        pkg image-update [-fnvq] [--be-name имя] [--no-refresh] [--no-"
"index]\n"
"        pkg refresh [--full] [publisher ...]\n"
"        pkg version\n"
"\n"
"Дополнительные субкоманды:\n"
"        pkg info [-lr] [--license] [pkg_fmri_pattern ...]\n"
"        pkg search [-alprI] [-s server] query\n"
"        pkg verify [-Hqv] [pkg_fmri_pattern ...]\n"
"        pkg fix [pkg_fmri_pattern ...]\n"
"        pkg contents [-Hmr] [-o attribute ...] [-s sort_key]\n"
"            [-t action_type ... ] [pkg_fmri_pattern ...]\n"
"        pkg image-create [-fFPUz] [--force] [--full|--partial|--user] [--"
"zone]\n"
"            [-k ssl_key] [-c ssl_cert] [--no-refresh] \n"
"            [--variant <variant_spec>=<instance>]\n"
"            (-p|--publisher) name=uri dir\n"
"\n"
"        pkg set-property propname propvalue\n"
"        pkg unset-property propname ...\n"
"        pkg property [-H] [propname ...]\n"
"\n"
"        pkg set-publisher [-Ped] [-k ssl_key] [-c ssl_cert]\n"
"            [-O origin_uri] [-m mirror_to_add | --add-mirror=mirror_to_add]\n"
"            [-M mirror_to_remove | --remove-mirror=mirror_to_remove]\n"
"            [--enable] [--disable] [--no-refresh] [--reset-uuid] publisher\n"
"        pkg unset-publisher publisher ...\n"
"        pkg publisher [-HPa] [publisher ...]\n"
"        pkg history [-Hl]\n"
"        pkg purge-history\n"
"        pkg rebuild-index\n"
"\n"
"Параметры:\n"
"        -R dir\n"
"        --help or -?\n"
"\n"
"Среда:\n"
"        PKG_IMAGE"

#: ../client.py:220
msgid "-s and -v may not be combined"
msgstr "опции -s и -v совместно использовать нельзя"

#: ../client.py:316 ../client.py:1148 ../client.py:1425
msgid "no packages installed"
msgstr "ни одного пакета не установлено"

#: ../client.py:323
msgid "No specified packages have available updates"
msgstr "Для указанных пакетов нет доступных обновлений"

#: ../client.py:326
msgid "No installed packages have available updates"
msgstr "Для установленных пакетов нет доступных обновлений"

#: ../client.py:352
#, python-format
msgid "no packages matching '%s' %s"
msgstr "Нет пакетов, соответствующих '%s' %s"

#: ../client.py:395
#, python-format
msgid "Created ZFS snapshot: %s"
msgstr "Создан снимок ZFS: %s"

#: ../client.py:425
msgid "verify: -v and -q may not be combined"
msgstr "verify: опции -v и -q совместно использовать нельзя"

#: ../client.py:480
msgid ""
"pkg: no packages matching the following patterns you specified are\n"
"installed on the system.\n"
msgstr ""
"pkg: в системе не установлен ни один пакет,\n"
"соответствующий указанным шаблонам.\n"

#: ../client.py:490
#, python-format
msgid ""
"\n"
"All other patterns matched installed packages.  %s"
msgstr ""
"\n"
"Для всех остальных шаблонов подобраны установленные пакеты.  %s"

#: ../client.py:531
msgid "image-update: -v and -q may not be combined"
msgstr "image-update: опции -v и -q совместно использовать нельзя"

#: ../client.py:534
#, python-format
msgid "image-update: command does not take operands ('%s')"
msgstr "image-update: команда не принимает операнды ('%s')"

#: ../client.py:544 ../client.py:587
#, python-format
msgid "No image rooted at '%s'"
msgstr "Нет ни одного образа в корневой папке '%s'"

#: ../client.py:558 ../client.py:726
msgid "No updates available for this image."
msgstr "Для данного образа нет доступных обновлений."

#: ../client.py:561
#, python-format
msgid ""
"image-update failed (inventory exception):\n"
"%s"
msgstr ""
"сбой выполнения команды image-update (исключение учета):\n"
"%s"

#: ../client.py:581
msgid ""
"WARNING: pkg(5) appears to be out of date, and should be updated before\n"
"running image-update.\n"
msgstr ""
"ВНИМАНИЕ: pkg(5) оказался устаревшим и должен быть обновлен\n"
"до выполнения команды image-update.\n"

#: ../client.py:583
msgid ""
"Please update pkg(5) using 'pfexec pkg install SUNWipkg' and then retry\n"
"the image-update."
msgstr ""
"Обновите pkg(5) командой 'pfexec pkg install SUNWipkg', а затем \n"
"выполните команду image-update еще раз."

#: ../client.py:611
msgid ""
"\n"
"An unexpected error happened while preparing for image-update:"
msgstr ""
"\n"
"Во время подготовки к выполнению команды image-update произошла неожиданная ошибка:"

#: ../client.py:618
#, python-format
msgid "image-update failed: %s"
msgstr "сбой команды image-update: %s"

#: ../client.py:621
msgid "image-update cannot be done on live image"
msgstr "команду image-update нельзя выполнять на используемом образе"

#: ../client.py:643
#, python-format
msgid ""
"\n"
"An unexpected error happened during image-update: %s"
msgstr ""
"\n"
"Во время выполнения команды image-update произошла неожиданная ошибка: %s"

#: ../client.py:649
msgid "NOTE: Please review release notes posted at:\n"
msgstr "ПРИМЕЧАНИЕ. Ознакомьтесь с информацией о выпуске по адресу:\n"

#: ../client.py:695
msgid "install: at least one package name required"
msgstr "install: требуется хотя бы одно название пакета"

#: ../client.py:698
msgid "install: -v and -q may not be combined"
msgstr "install: опции -v и -q совместно использовать нельзя"

#: ../client.py:713 ../client.py:849 ../client.py:1030 ../client.py:1612
#: ../client.py:1697 ../client.py:1821 ../client.py:1875 ../client.py:2261
#, python-format
msgid "'%s' is not an install image"
msgstr "'%s' не является образом установки"

#: ../client.py:745
#, python-format
msgid ""
"install failed (inventory exception):\n"
"%s"
msgstr ""
"сбой выполнения команды install (исключение учета):\n"
"%s"

#: ../client.py:771 ../client.py:890
msgid ""
"\n"
"An unexpected error happened while preparing for install:"
msgstr ""
"\n"
"Во время подготовки к установке произошла неожиданная ошибка:"

#: ../client.py:780
#, python-format
msgid "installation failed: %s"
msgstr "сбой установки: %s"

#: ../client.py:801
#, python-format
msgid "An unexpected error happened during installation: %s"
msgstr "Во время установки произошла неожиданная ошибка: %s"

#: ../client.py:831
msgid "uninstall: at least one package name required"
msgstr "uninstall: необходимо хотя бы одно название пакета"

#: ../client.py:834
msgid "uninstall: -v and -q may not be combined"
msgstr "uninstall: опции -v и -q совместно использовать нельзя"

#: ../client.py:857
#, python-format
msgid ""
"uninstall failed (inventory exception):\n"
"%s"
msgstr ""
"сбой выполнения команды uninstall (исключение учета):\n"
"%s"

#: ../client.py:899
#, python-format
msgid "uninstallation failed: %s"
msgstr "сбой удаления: %s"

#: ../client.py:918
#, python-format
msgid "An unexpected error happened during uninstallation: %s"
msgstr "Во время удаления произошла неожиданная ошибка: %s"

#: ../client.py:938
#, python-format
msgid ""
"The server returned a malformed result.\n"
"The problematic structure: %r"
msgstr ""
"Сервер вернул недопустимый результат.\n"
"Структура, вызвавшая проблему: %r"

#: ../client.py:959
#, python-format
msgid ""
"The server returned a malformed result.\n"
"The problematic structure:%r"
msgstr ""
"Сервер вернул недопустимый результат.\n"
"Структура, вызвавшая проблему: %r"

#: ../client.py:969
#, python-format
msgid ""
"The server returned a malformed action.\n"
"%s"
msgstr ""
"Сервер вернул недопустимое действие.\n"
"%s"

#: ../client.py:1008
#, python-format
msgid "%s is not a valid server URL."
msgstr "%s не является действительным адресом URL сервера"

#: ../client.py:1058
#, python-format
msgid "The server returned a malformed result:%r"
msgstr "Сервер вернул недопустимый результат:%r"

#: ../client.py:1072 ../client.py:1082
msgid ""
"The search index appears corrupted.  Please rebuild the index with 'pkg "
"rebuild-index'."
msgstr ""
"Индекс поиска представляется поврежденным. Выполните перестроение индекса с помощью команды 'pkg "
"rebuild-index'."

#: ../client.py:1116
msgid "info: -l and -r may not be combined"
msgstr "info: опции -l и -r совместно использовать нельзя"

#: ../client.py:1119
msgid "info: must request remote info for specific packages"
msgstr "info: для определенных пакетов необходимо запрашивать сведения удаленно "

#: ../client.py:1165 ../gui/data/packagemanager.glade.h:50
#: ../packagemanager.py:1250
msgid "Installed"
msgstr "Установлено"

#: ../client.py:1167
msgid "Not installed"
msgstr "Не установлено"

#: ../client.py:1171
msgid "          Name:"
msgstr "                   Имя:"

#: ../client.py:1173
msgid "       Summary:"
msgstr "     Заключение:"

#: ../client.py:1176
msgid "      Category:"
msgstr "        Категория:"

#: ../client.py:1183
msgid "         State:"
msgstr "        Состояние:"

#. XXX even more info on the publisher would be nice?
#: ../client.py:1186
msgid "     Publisher:"
msgstr "     Издатель:"

#: ../client.py:1187
msgid "       Version:"
msgstr "              Версия:"

#: ../client.py:1188
msgid " Build Release:"
msgstr " Версия сборки:"

#: ../client.py:1189
msgid "        Branch:"
msgstr "                Ветвь:"

#: ../client.py:1190
msgid "Packaging Date:"
msgstr " Дата упаковки:"

#: ../client.py:1191
msgid "          Size:"
msgstr "             Размер:"

#: ../client.py:1192
msgid "          FMRI:"
msgstr "                 FMRI:"

#: ../client.py:1200 ../client.py:1530
msgid ""
"pkg: no packages matching the following patterns you specified are\n"
"installed on the system.  Try specifying -r to query remotely:"
msgstr ""
"pkg: в системе не установлен ни один пакет,\n"
"соответствующий указанным шаблонам. Попробуйте указать опцию -r, чтобы удаленно запросить:"

#: ../client.py:1204 ../client.py:1534
msgid ""
"pkg: no packages matching the following patterns you specified were\n"
"found in the catalog.  Try relaxing the patterns, refreshing, and/or\n"
"examining the catalogs:"
msgstr ""
"pkg: в каталоге не найден ни один пакет, соответствующий\n"
"указанным шаблонам. Попробуйте сократить шаблоны, обновить и /или\n"
"осмотреть каталоги:"

#: ../client.py:1220 ../modules/client/api_errors.py:154
#, python-format
msgid "'%s' matches multiple packages"
msgstr "'%s' соответствует нескольким пакетам"

#: ../client.py:1226
msgid "no license information could be found for the following packages:"
msgstr "для следующих пакетов не обнаружена информация о лицензии:"

#: ../client.py:1381
msgid "contents: -l and -r may not be combined"
msgstr "contents: опции -l и -r совместно использовать нельзя"

#: ../client.py:1384
msgid "contents: must request remote contents for specific packages"
msgstr "contents: для определенных пакетов необходимо запрашивать содержание удаленно"

#: ../client.py:1398
#, python-format
msgid "contents: -m and %s may not be specified at the same time"
msgstr "contents: опции -m и %s нельзя указывать одновременно"

#: ../client.py:1403 ../client.py:1406
#, python-format
msgid "Invalid attribute '%s'"
msgstr "Недопустимый атрибут '%s'"

#: ../client.py:1466 ../client.py:1472
#, python-format
msgid "pkg: '%s' matches multiple packages"
msgstr "pkg: '%s' соответствует нескольким пакетам"

#: ../client.py:1550
#, python-format
msgid "pkg: %s/%s catalogs successfully updated:"
msgstr "pkg: каталоги %s/%s успешно обновлены:"

#: ../client.py:1578
#, python-format
msgid "Could not update catalog for '%s' due to insufficient permissions."
msgstr "Недостаточно полномочий для обновления каталога '%s'."

#: ../client.py:1582
msgid "Could not update a catalog due to insufficient permissions."
msgstr "Недостаточно полномочий для обновления каталога."

#: ../client.py:1585
msgid ""
"Please try the command again using pfexec, or otherwise increase \n"
"    your permissions."
msgstr ""
"Повторите попытку выполнения команды с помощью pfexec, либо повысьте уровень полномочий \n"
"    другим способом."

#: ../client.py:1622
msgid "'pkg publisher' will show a list of publishers."
msgstr "Команда 'pkg publisher' выводит список издателей."

#: ../client.py:1680
msgid "requires a publisher name"
msgstr "требуется имя издателя"

#: ../client.py:1682
msgid "only one publisher name may be specified"
msgstr "можно ввести только одно имя издателя"

#: ../client.py:1688
msgid "the -p and -d options may not be combined"
msgstr "не допускается совместное использование параметров -p и -d"

#: ../client.py:1712
msgid ""
"publisher does not exist. Use -O to define origin URI for new publisher."
msgstr ""
"издатель не существует. Определите URI источника для нового издателя параметром  -O."

#: ../client.py:1794
#, python-format
msgid ""
"The origin URIs for '%s' do not appear to point to a valid pkg server.\n"
"Please check the server's address and client's network configuration.\n"
"Additional details:\n"
"\n"
"%s"
msgstr ""
"Идентификаторы URI источников для '%s' не указывают на действительный сервер pkg.\n"
"Проверьте адрес сервера и параметры настройки сети клиента.\n"
"Дополнительные сведения:\n"
"\n"
"%s"

#: ../client.py:1847
#, python-format
msgid "Removal failed for '%(pub)s': %(msg)s"
msgstr "Неудачное удаление для '%(pub)s': %(msg)s"

#: ../client.py:1919
msgid "PUBLISHER"
msgstr "PUBLISHER"

#: ../client.py:1919
msgid "TYPE"
msgstr "TYPE"

#: ../client.py:1919
msgid "STATUS"
msgstr "STATUS"

#: ../client.py:1920
msgid "URI"
msgstr "URI"

#: ../client.py:1935
msgid "(preferred)"
msgstr "(предпочтительно)"

#: ../client.py:1937
msgid "(disabled)"
msgstr "(отменены)"

#. XXX get the real origin status
#: ../client.py:1944
msgid "origin"
msgstr "источник"

#. XXX get the real mirror status
#: ../client.py:1948
msgid "mirror"
msgstr "зеркало"

#: ../client.py:1954
msgid "              SSL Key:"
msgstr "              Ключ SSL:"

#: ../client.py:1955
msgid "             SSL Cert:"
msgstr "             Сертификат SSL:"

#: ../client.py:1967
msgid " Cert. Effective Date:"
msgstr " Дата вступления в силу сертификата:"

#: ../client.py:1969
msgid "Cert. Expiration Date:"
msgstr "Дата истечения действия сертификата:"

#: ../client.py:1976
msgid "           Origin URI:"
msgstr "           URI источника:"

#: ../client.py:1982
msgid "           Mirror URI:"
msgstr "           URI зеркала:"

#: ../client.py:1996
msgid "            Publisher:"
msgstr "            Издатель:"

#: ../client.py:1997
msgid "                Alias:"
msgstr "                Псевдоним:"

#: ../client.py:2007
msgid "          Client UUID:"
msgstr "          UUID клиента:"

#: ../client.py:2008
msgid "      Catalog Updated:"
msgstr "      Каталог обновлен:"

#: ../client.py:2010 ../client.py:2012
msgid "              Enabled:"
msgstr "              Включен:"

#: ../client.py:2010
msgid "No"
msgstr "Нет"

#: ../client.py:2012
msgid "Yes"
msgstr "Да"

#: ../client.py:2023
msgid "set-property: requires a property name and value"
msgstr "set-property: необходимо название и значение свойства"

#: ../client.py:2026
msgid ""
"set-property: set-publisher must be used to change the preferred publisher"
msgstr ""
"Для изменения предпочтительного издателя следует использовать свойство set-property: set-publisher"

#: ../client.py:2050
msgid "unset-property: requires at least one property name"
msgstr "unset-property: для команды необходимо хотя бы одно название свойства"

#: ../client.py:2054
msgid ""
"unset-property: set-publisher must be used to change the preferred publisher"
msgstr ""
"Для изменения предпочтительного издателя следует использовать свойство unset-property: set-publisher"

#: ../client.py:2061
#, python-format
msgid "unset-property: no such property: %s"
msgstr "unset-property: нет такого свойства: %s"

#: ../client.py:2082
#, python-format
msgid "property: no such property: %s"
msgstr "property: нет такого свойства: %s"

#: ../client.py:2145 ../client.py:2180
msgid ""
"image-create requires publisher argument to be of the form '<prefix>=<url>'."
msgstr ""
"для команды image-create необходимо, чтобы аргумент publisher имел вид '<prefix>=<url>'."

#: ../client.py:2154
msgid ""
"image-create requires variant arguments to be of the form '<name>=<value>'."
msgstr ""
"для команды image-create необходимо, чтобы аргумент variant имел вид '<name>=<value>'."

#: ../client.py:2160
msgid "image-create requires a single image directory path"
msgstr "для команды image-create необходим путь одного изображения "

#: ../client.py:2165
#, python-format
msgid "pkg: set-publisher: SSL key file '%s' does not exist"
msgstr "pkg: set-publisher: файл ключа SSL '%s' не существует"

#: ../client.py:2172
#, python-format
msgid "pkg: set-publisher: SSL key cert '%s' does not exist"
msgstr "pkg: set-publisher: сертификат ключа SSL '%s' не существует"

#: ../client.py:2177
msgid "image-create requires a publisher argument"
msgstr "для команды image-create необходим аргумент publisher"

#: ../client.py:2184
#, python-format
msgid "image-create requires that a prefix not match: %s"
msgstr "для команды image-create необходимо, чтобы префикс не соответствовал: %s"

#: ../client.py:2189
msgid "image-create: publisher prefix has invalid characters"
msgstr "image-create: недопустимые символы в префиксе publisher"

#: ../client.py:2194
msgid "image-create: publisher URI is invalid"
msgstr "image-create: недопустимый идентификатор URI издателя"

#: ../client.py:2201
#, python-format
msgid "there is already an image at: %s"
msgstr "образ уже находится по адресу: %s"

#: ../client.py:2202 ../client.py:2209
msgid "To override, use the -f (force) option."
msgstr "Для переопределения используйте параметр -f (принудительно)."

#: ../client.py:2208
#, python-format
msgid "Non-empty directory: %s"
msgstr "Каталог не пуст: %s"

#: ../client.py:2221
#, python-format
msgid "cannot create image at %s: %s"
msgstr "невозможно создать образ по адресу %s: %s"

#: ../client.py:2232
#, python-format
msgid ""
"The URI '%s' does not appear to point to a valid pkg server.\n"
"Please check the server's address and client's network configuration.\n"
"Additional details:\n"
"\n"
"%s"
msgstr ""
"Идентификатор URI источника для '%s' не указывает на действительный сервер pkg.\n"
"Проверьте адрес сервера и параметры настройки сети клиента.\n"
"Дополнительные сведения:\n"
"\n"
"%s"

#: ../client.py:2255
#, python-format
msgid "rebuild-index: command does not take operands ('%s')"
msgstr "rebuild-index: команда не принимает операнды ('%s')"

#: ../client.py:2293
msgid "history: -H and -l may not be combined"
msgstr "history: опции -H и -l совместно использовать нельзя"

#: ../client.py:2297
msgid "TIME"
msgstr "ВРЕМЯ"

#: ../client.py:2298
msgid "OPERATION"
msgstr "РАБОТА"

#: ../client.py:2298
msgid "CLIENT"
msgstr "КЛИЕНТ"

#: ../client.py:2298
msgid "OUTCOME"
msgstr "РЕЗУЛЬТАТ"

#: ../client.py:2391
#, python-format
msgid "illegal global option -- %s"
msgstr "недопустимая глобальная опция -- %s"

#: ../client.py:2399
#, python-format
msgid "%(opt)s takes argument of form name=value, not %(arg)s"
msgstr "%(opt)s принимает аргумент в форме название=значение, а не %(arg)s"

#: ../client.py:2433 ../client.py:2443
#, python-format
msgid "-R not allowed for %s subcommand"
msgstr "-R не разрешен для субкоманды %s"

#: ../client.py:2438 ../client.py:2544 ../publish.py:355
#, python-format
msgid "illegal %s option -- %s"
msgstr "недопустимая опция %s -- %s"

#: ../client.py:2446
#, python-format
msgid "version: command does not take operands ('%s')"
msgstr "version: команда не принимает операнды ('%s')"

#: ../client.py:2471
msgid ""
"Could not find image.  Use the -R option or set $PKG_IMAGE to point\n"
"to an image, or change the working directory to one inside the image."
msgstr ""
"Образ не найден. Используйте опцию -R или настройте $PKG_IMAGE для указания\n"
"образа, или измените рабочий каталог на каталог внутри образа."

#: ../client.py:2485
msgid "No image found."
msgstr "Не найден ни один образ."

#: ../client.py:2491
#, python-format
msgid "client configuration error: %s"
msgstr "ошибка настройки клиента: %s"

#: ../client.py:2536
msgid "History purged."
msgstr "История очищена."

#: ../client.py:2541 ../publish.py:353
#, python-format
msgid "unknown subcommand '%s'"
msgstr "неизвестная субкоманда '%s'"

#: ../client.py:2590
#, python-format
msgid ""
"\n"
"Maximum number of network retries exceeded during download. Details follow:\n"
"%s"
msgstr ""
"\n"
"Во время загрузки было превышено максимальное количество попыток доступа к сети. Подробные сведения см. ниже:\n"
"%s"

#: ../client.py:2597
msgid ""
"An error was encountered while attempting to retrieve package or file data "
"for the requested operation."
msgstr ""
"Произошла ошибка во время попытки извлечения пакета или данных из файла "
"для запрошенной операции."

#: ../client.py:2604
msgid ""
"\n"
"Unable to contact a valid package depot. This may be due to a problem with "
"the server, network misconfiguration, or an incorrect pkg client "
"configuration.  Please check your network settings and attempt to contact "
"the server using a web browser."
msgstr ""
"\n"
"Невозможно подключиться к действительному хранилищу пакетов. Возможны проблемы с"
"сервером, неверная настройка сети, либо неверная настройка клиента pkg"
"Проверьте параметры настройки сети и повторите попытку связи"
"с сервером с помощью обозревателя."

#: ../client.py:2609
#, python-format
msgid ""
"\n"
"Additional details:\n"
"\n"
"%s"
msgstr ""
"\n"
"Дополнительные сведения:\n"
"\n"
"%s"

#: ../client.py:2616
msgid ""
"An error was encountered while attempting to load history information\n"
"about past client operations."
msgstr ""
"Произошла ошибка во время попытки загрузки информации\n"
"из истории последних операций клиента."

#: ../client.py:2625
msgid ""
"An error was encountered while attempting to store information about the\n"
"current operation in client history."
msgstr ""
"Произошла ошибка во время попытки сохранения в истории клиента\n"
"информации о текущей операции."

#: ../client.py:2635
msgid "An error was encountered while attempting to purge client history."
msgstr " Произошла ошибка во время попытки очистки истории клиента."

#: ../client.py:2642
#, python-format
msgid ""
"The pkg command appears out of sync with the libraries provided \n"
"by SUNWipkg. The client version is %(client)s while the library API version "
"is %(api)s"
msgstr ""
"Команда pkg не синхронизирована с библиотеками, поставляемыми \n"
"SUNWipkg. Версия клиента -  %(client)s, а версия  программного интерфейса "
"приложения библиотеки - %(api)s"

#: ../client.py:2654 ../publish.py:379
#, python-format
msgid ""
"\n"
"\n"
"This is an internal error.  Please let the developers know about this\n"
"problem by filing a bug at http://defect.opensolaris.org and including the\n"
"above traceback and this message.  The version of pkg(5) is '%s'."
msgstr ""
"\n"
"\n"
"Это внутренняя ошибка. Сообщите разработчикам об этой неполадке\n"
"заполнив отчет по адресу http://defect.opensolaris.org и включив\n"
"результат отслеживания причины неполадки и данное сообщение. Версия pkg(5): '%s'."

#: ../gui/data/l10n.py:31
msgid "AMP Stack"
msgstr "Стек AMP"

#: ../gui/data/l10n.py:32
msgid "Accessories"
msgstr "Стандартные приложения"

#: ../gui/data/l10n.py:33
msgid "Administration and Configuration"
msgstr "Администрирование и настройка"

#: ../gui/data/l10n.py:34
msgid "Application and Web Servers"
msgstr "Приложение и веб-серверы"

#: ../gui/data/l10n.py:35
msgid "Builds"
msgstr "Компоновки"

#: ../gui/data/l10n.py:36
msgid "C"
msgstr "C"

#: ../gui/data/l10n.py:37
msgid "C++"
msgstr "C++"

#: ../gui/data/l10n.py:38
msgid "Communications"
msgstr "Средства связи"

#: ../gui/data/l10n.py:39
msgid "Configuration and Preferences"
msgstr "Настройка и параметры"

#: ../gui/data/l10n.py:40
msgid "Core"
msgstr "Ядро"

#: ../gui/data/l10n.py:41
msgid "Databases"
msgstr "Базы данных"

#: ../gui/data/l10n.py:42
msgid "Databases (System)"
msgstr "Базы данных (система)"

#: ../gui/data/l10n.py:43
msgid "Developer Tools"
msgstr "Инструменты разработчика"

#: ../gui/data/l10n.py:44
msgid "Display"
msgstr "Дисплей"

#: ../gui/data/l10n.py:45
msgid "Distribution Tools"
msgstr "Средства дистрибутива"

#: ../gui/data/l10n.py:46
msgid "Documentation"
msgstr "Документация"

#: ../gui/data/l10n.py:47
msgid "Editors"
msgstr "Редакторы"

#: ../gui/data/l10n.py:48
msgid "Enterprise Management"
msgstr "Управление предприятием"

#: ../gui/data/l10n.py:49
msgid "File Managers"
msgstr "Диспетчеры файлов"

#: ../gui/data/l10n.py:50
msgid "File System"
msgstr "Файловая система"

#: ../gui/data/l10n.py:51
msgid "Fonts"
msgstr "Шрифты"

#: ../gui/data/l10n.py:52
msgid "GNOME and GTK+"
msgstr "GNOME и GTK+"

#: ../gui/data/l10n.py:53
msgid "GNU"
msgstr "GNU"

#: ../gui/data/l10n.py:54
msgid "Games"
msgstr "Игры"

#: ../gui/data/l10n.py:55
msgid "Graphics and Imaging"
msgstr "Графика и работа с изображениями"

#: ../gui/data/l10n.py:56
msgid "Hardware"
msgstr "Оборудование"

#: ../gui/data/l10n.py:57
msgid "High Performance Computing"
msgstr "Высокопроизводительные вычисления"

#: ../gui/data/l10n.py:58
msgid "Integrated Development Environments"
msgstr "Интегрированные среды разработки"

#: ../gui/data/l10n.py:59
msgid "Internationalization"
msgstr "Интернационализация"

#: ../gui/data/l10n.py:60
msgid "Internet"
msgstr "Интернет"

#: ../gui/data/l10n.py:61
msgid "Java"
msgstr "Java"

#: ../gui/data/l10n.py:62
msgid "Libraries"
msgstr "Библиотеки"

#: ../gui/data/l10n.py:63
msgid "Libraries (System)"
msgstr "Библиотеки (система)"

#: ../gui/data/l10n.py:64
msgid "Localizations (Desktop)"
msgstr "Локализация (рабочий стол)"

#: ../gui/data/l10n.py:65
msgid "Localizations (System)"
msgstr "Локализация (система)"

#: ../gui/data/l10n.py:66
msgid "Media"
msgstr "Мультимедиа"

#: ../gui/data/l10n.py:67
msgid "Multimedia Libraries"
msgstr "Мультимедийные библиотеки"

#: ../gui/data/l10n.py:68
msgid "Networking"
msgstr "Подключение к сети"

#: ../gui/data/l10n.py:69
msgid "Office"
msgstr "Офис"

#: ../gui/data/l10n.py:70
msgid "Office Tools"
msgstr "Офисные инструменты"

#: ../gui/data/l10n.py:71
msgid "Other Languages"
msgstr "Другие языки"

#: ../gui/data/l10n.py:72
msgid "Other Peripherals"
msgstr "Другие периферийные устройства"

#: ../gui/data/l10n.py:73
msgid "PHP"
msgstr "PHP"

#: ../gui/data/l10n.py:74
msgid "Packaging"
msgstr "Упаковка"

#: ../gui/data/l10n.py:75
msgid "Panels and Applets"
msgstr "Панели и апплеты"

#: ../gui/data/l10n.py:76
msgid "Perl"
msgstr "Perl"

#: ../gui/data/l10n.py:77
msgid "Plug-ins and Run-times"
msgstr "Дополнения и время выполнения"

#: ../gui/data/l10n.py:78
msgid "Ports"
msgstr "Порты"

#: ../gui/data/l10n.py:79
msgid "Printing"
msgstr "Печать"

#: ../gui/data/l10n.py:80
msgid "Python"
msgstr "Python"

#: ../gui/data/l10n.py:81
msgid "Releases"
msgstr "Выпуски"

#: ../gui/data/l10n.py:82
msgid "Ruby"
msgstr "Ruby"

#: ../gui/data/l10n.py:83
msgid "Scripts"
msgstr "Сценарии"

#: ../gui/data/l10n.py:84
msgid "Security"
msgstr "Безопасность"

#: ../gui/data/l10n.py:85
msgid "Services"
msgstr "Службы"

#: ../gui/data/l10n.py:86
msgid "Sessions"
msgstr "Сеансы"

#: ../gui/data/l10n.py:87
msgid "Shells"
msgstr "Интерпретаторы команд"

#: ../gui/data/l10n.py:88
msgid "Software Management"
msgstr "Управление программным обеспечением"

#: ../gui/data/l10n.py:89
msgid "Sound and Video"
msgstr "Звук и видео"

#: ../gui/data/l10n.py:90
msgid "Source Code Management"
msgstr "Управление исходным кодом"

#: ../gui/data/l10n.py:91
msgid "Storage"
msgstr "Хранилище"

#: ../gui/data/l10n.py:92 ../packagemanager.py:1352
msgid "System"
msgstr "Система"

#: ../gui/data/l10n.py:93
msgid "System Utilities"
msgstr "Системные служебные программы"

#: ../gui/data/l10n.py:94
msgid "Text Tools"
msgstr "Текстовые средства"

#: ../gui/data/l10n.py:95
msgid "Theming"
msgstr "Темы"

#: ../gui/data/l10n.py:96
msgid "Trusted"
msgstr "Надежный"

#: ../gui/data/l10n.py:97
msgid "Trusted Extensions"
msgstr "Trusted Extensions"

#: ../gui/data/l10n.py:98
msgid "Universal Access"
msgstr "Универсальный доступ"

#: ../gui/data/l10n.py:99
msgid "Virtualization"
msgstr "Виртуализация"

#: ../gui/data/l10n.py:100
msgid "Window Managers"
msgstr "Диспетчеры окон"

#: ../gui/data/l10n.py:101
msgid "X11"
msgstr "X11"

#: ../gui/data/l10n.py:102
msgid "X11 (System)"
msgstr "X11 (система)"

#: ../gui/data/addmoresoftware.desktop.in.h:1
msgid "Add More Software"
msgstr "Добавить дополнительное программное обеспечение"

#: ../gui/data/addmoresoftware.desktop.in.h:2
#: ../gui/data/packagemanager.desktop.in.h:1
msgid "Install, update and remove software packages"
msgstr "Установить, обновить или удалить программные пакеты"

#. XXX Remove and use _() where self._ and self.parent._ are being used
#: ../gui/data/packagemanager.desktop.in.h:2
#: ../gui/data/packagemanager.glade.h:58 ../gui/modules/webinstall.py:315
#: ../packagemanager.py:218 ../packagemanager.py:2208
#: ../packagemanager.py:2218 ../packagemanager.py:3357
msgid "Package Manager"
msgstr "Диспетчер пакетов"

#: ../gui/data/packagemanager.glade.h:1
msgid "<b>Add new repositories and manage existing ones.</b>"
msgstr "<b>Добавление новых хранилиз и управление существующими.</b>"

#: ../gui/data/packagemanager.glade.h:2
msgid "<b>Apply the following changes:</b>"
msgstr "<b>Применить следующие изменения:</b>"

#: ../gui/data/packagemanager.glade.h:3
msgid ""
"<b>Create New Boot Environment</b>\n"
"\n"
"Update All creates a new boot environment (BE) as a means of archiving the "
"current BE. The new BE consists of the updated packages and is the default "
"BE when the system is rebooted.\n"
msgstr ""
"<b>Создание новой среды загрузки</b>\n"
"\n"
"Команда Update All приводит к созданию новой среды загрузки в целях архивации "
"текущей среды загрузки. Новая среда загрузки состоит из обновленных пакетов"
"и используется по умолчанию при загрузке системы.\n"

#: ../gui/data/packagemanager.glade.h:9
msgid "<b>Delete, rename and activate boot environments.</b>"
msgstr "<b>Удаление, переименование и активация сред загрузки.</b>"

#: ../gui/data/packagemanager.glade.h:10
msgid "<b>Modify Repository URL</b>"
msgstr "<b>Изменить адрес URL хранилища</b>"

#: ../gui/data/packagemanager.glade.h:11
msgid "<b>PACKAGE NAME</b>"
msgstr "<b>НАЗВАНИЕ ПАКЕТА</b>"

#: ../gui/data/packagemanager.glade.h:12
msgid "<b>Package Name</b>"
msgstr "<b>Название пакета</b>"

#: ../gui/data/packagemanager.glade.h:13
msgid "<b>Review the packages to be removed:</b>"
msgstr "<b>Просмотр удаляемых пакетов:</b>"

#: ../gui/data/packagemanager.glade.h:14
msgid "<b>Search</b>"
msgstr "<b>Поиск</b>"

#: ../gui/data/packagemanager.glade.h:15
msgid "<b>Startup</b>"
msgstr "<b>Запуск</b>"

#: ../gui/data/packagemanager.glade.h:16
msgid "<b>_Mirrors</b>"
msgstr "<b>_Зеркала</b>"

#: ../gui/data/packagemanager.glade.h:17
msgid "<b>_SSL Key and Certificate</b>"
msgstr "<b>_Ключ и сертификат SSL</b>"

#: ../gui/data/packagemanager.glade.h:18
msgid "<big><b>Package Manager Web Installer</b></big>"
msgstr "<big><b>Веб-установщик диспетчера пакетов</b></big>"

#: ../gui/data/packagemanager.glade.h:19
msgid "<i>_Repository:</i>"
msgstr "<i>_Хранилище:</i>"

#: ../gui/data/packagemanager.glade.h:20
msgid "About"
msgstr "О программе"

#: ../gui/data/packagemanager.glade.h:21
msgid "About Glade"
msgstr "О Glade"

#: ../gui/data/packagemanager.glade.h:22
msgid "B_rowse"
msgstr "_Обзор"

#: ../gui/data/packagemanager.glade.h:23
msgid "Boot Environment Confirmation"
msgstr "Подтверждение среды загрузки"

#: ../gui/data/packagemanager.glade.h:24
msgid "Br_owse"
msgstr "О_бзор"

#: ../gui/data/packagemanager.glade.h:25
msgid "Browse for SSL Certificate"
msgstr "Выбор сертификата SSL"

#: ../gui/data/packagemanager.glade.h:26
msgid "Browse for SSL Key"
msgstr "Выбор ключа SSL"

#: ../gui/data/packagemanager.glade.h:27 ../um/data/updatemanager.glade.h:4
msgid "Checking for latest SUNWipkg and SUNWipkg-gui, please wait..."
msgstr "Проверка новейших SUNWipkg и SUNWipkg-gui, подождите..."

#: ../gui/data/packagemanager.glade.h:28
msgid "Checking package dependencies..."
msgstr "Проверка зависимостей пакета..."

#: ../gui/data/packagemanager.glade.h:29
msgid "Clear"
msgstr "Очистить"

#: ../gui/data/packagemanager.glade.h:30
msgid "Click to get list of newest packages."
msgstr "Выберите для получения списка новых пакетов."

#: ../gui/data/packagemanager.glade.h:31
msgid "Click to update all installed packages."
msgstr "Выберите для обновления всех установленных пакетов."

#: ../gui/data/packagemanager.glade.h:32
msgid "Copy"
msgstr "Копировать"

#: ../gui/data/packagemanager.glade.h:33
msgid ""
"Copyright 2009 Sun Microsystems, Inc.  \n"
"All rights reserved.\n"
"Use is subject to license terms."
msgstr ""
"Авторское право 2009, Sun Microsystems, Inc. \n"
"Все права защищены.\n"
"Использование регулируется лицензионным соглашением."

#: ../gui/data/packagemanager.glade.h:36
msgid "Cu_t"
msgstr "_Вырезать"

#: ../gui/data/packagemanager.glade.h:37
msgid "Currently active boot environment"
msgstr "Активная в данный момент загрузочная среда"

#: ../gui/data/packagemanager.glade.h:38
msgid "Cut action"
msgstr "Вырезать действие"

#: ../gui/data/packagemanager.glade.h:39
msgid "Dependencies"
msgstr "Зависимости"

#: ../gui/data/packagemanager.glade.h:40
msgid "Deselect selected packages"
msgstr "Отменить выбор пакетов"

#: ../gui/data/packagemanager.glade.h:41
msgid "Details"
msgstr "Подробности"

#: ../gui/data/packagemanager.glade.h:42
msgid "Do not notify me about these repositories again."
msgstr "Более не уведомлять об этих хранилищах."

#: ../gui/data/packagemanager.glade.h:43 ../gui/modules/installupdate.py:101
#: ../um/data/updatemanager.glade.h:6 ../updatemanager.py:175
#: ../updatemanager.py:427 ../updatemanager.py:434 ../updatemanager.py:441
#: ../updatemanager.py:448 ../updatemanager.py:455
msgid "Download"
msgstr "Загрузка"

#: ../gui/data/packagemanager.glade.h:44
msgid "Files"
msgstr "Файлы"

#: ../gui/data/packagemanager.glade.h:45
msgid "Help"
msgstr "Справка"

#: ../gui/data/packagemanager.glade.h:46 ../gui/modules/installupdate.py:102
#: ../updatemanager.py:191 ../updatemanager.py:428 ../updatemanager.py:435
#: ../updatemanager.py:442 ../updatemanager.py:449 ../updatemanager.py:456
msgid "Install"
msgstr "Установить"

#: ../gui/data/packagemanager.glade.h:47
msgid "Install or Update selected packages"
msgstr "Установка или обновление выбранных пакетов"

#: ../gui/data/packagemanager.glade.h:48 ../gui/modules/installupdate.py:238
msgid "Install/Update"
msgstr "Установка/обновление"

#: ../gui/data/packagemanager.glade.h:49
msgid "Install/Upgrade Check"
msgstr "Проверка установки/обновления"

#: ../gui/data/packagemanager.glade.h:51
msgid "Legend:"
msgstr "Условные обозначения:"

#: ../gui/data/packagemanager.glade.h:52
msgid "License"
msgstr "Лицензия"

#: ../gui/data/packagemanager.glade.h:53
msgid "M_irror:"
msgstr "_Зеркало:"

#: ../gui/data/packagemanager.glade.h:54
msgid "Manage Boot Environments"
msgstr "Управление загрузочными средами"

#: ../gui/data/packagemanager.glade.h:55
msgid "Manage Repositories"
msgstr "Управление хранилищами"

#: ../gui/data/packagemanager.glade.h:56
msgid "Manage _Boot Environments..."
msgstr "_Управление загрузочными средами..."

#: ../gui/data/packagemanager.glade.h:57
msgid "Modify Repository"
msgstr "Изменить хранилище"

#: ../gui/data/packagemanager.glade.h:59
msgid "Package Manager Preferences"
msgstr "Параметры диспетчера пакетов"

#: ../gui/data/packagemanager.glade.h:60
msgid "Paste"
msgstr "Вставить"

#: ../gui/data/packagemanager.glade.h:61
msgid "Pre_ferences"
msgstr "_Параметры"

#: ../gui/data/packagemanager.glade.h:62
msgid "Preferences"
msgstr "Параметры"

#: ../gui/data/packagemanager.glade.h:63 ../gui/modules/installupdate.py:100
msgid "Preparation"
msgstr "Подготовка"

#: ../gui/data/packagemanager.glade.h:64 ../gui/modules/webinstall.py:206
msgid "Proceed only if you trust this new repository "
msgstr "Выполнение можно продолжать, только если хранилище заслуживает доверия "

#: ../gui/data/packagemanager.glade.h:65 ../um/data/updatemanager.glade.h:9
msgid "Processing"
msgstr "Обработка"

#: ../gui/data/packagemanager.glade.h:66
msgid "Quit application"
msgstr "Выйти из приложения"

#: ../gui/data/packagemanager.glade.h:67
msgid "Reload"
msgstr "Перезагрузить"

#: ../gui/data/packagemanager.glade.h:68
msgid "Remote search error"
msgstr "Ошибка удаленного поиска"

#: ../gui/data/packagemanager.glade.h:69 ../gui/modules/installupdate.py:203
#: ../gui/modules/installupdate.py:206
msgid "Remove"
msgstr "Удалить"

#: ../gui/data/packagemanager.glade.h:70
msgid "Remove Confirmation"
msgstr "Удалить подтверждение"

#: ../gui/data/packagemanager.glade.h:71
msgid "Remove selected packages"
msgstr "Удалить выбранные пакеты"

#: ../gui/data/packagemanager.glade.h:72
msgid "SSL C_ertificate: "
msgstr "Сертификат _SSL "

#: ../gui/data/packagemanager.glade.h:73
msgid "SSL Key and Certificate"
msgstr "Ключ и сертификат SSL"

#: ../gui/data/packagemanager.glade.h:74
msgid "SSL _Certificate:"
msgstr "С_ертификат SSL"

#: ../gui/data/packagemanager.glade.h:75
msgid "SSL _Key:"
msgstr "_Ключ SSL:"

#: ../gui/data/packagemanager.glade.h:76
msgid "SSL _Key: "
msgstr "_Ключ SSL: "

#: ../gui/data/packagemanager.glade.h:77
msgid "Se_arch:"
msgstr "П_оиск:"

#: ../gui/data/packagemanager.glade.h:78
msgid "Search"
msgstr "Поиск"

#: ../gui/data/packagemanager.glade.h:79
msgid "Search All Repositories Error"
msgstr "Ошибка поиска по всем хранилищам"

#: ../gui/data/packagemanager.glade.h:80
msgid ""
"Search All Repositories cannot be performed on the following\n"
"repositories:"
msgstr ""
"Поиск по всем хранилищаи невозможен для следующих\n"
"хранилищ:"

#: ../gui/data/packagemanager.glade.h:82
msgid "Select All Items"
msgstr "Выбрать все элементы"

#: ../gui/data/packagemanager.glade.h:83
msgid "Select Updates"
msgstr "Выбор обновлений"

#: ../gui/data/packagemanager.glade.h:84
msgid "Select _All"
msgstr "В_ыбрать все"

#: ../gui/data/packagemanager.glade.h:85
msgid "Select _Updates"
msgstr "Выбор о_бновлений"

#: ../gui/data/packagemanager.glade.h:86
msgid "Select from the list to modify or remove existing repositories."
msgstr "В списке выберите изменить или удалить существующие хранилища."

#: ../gui/data/packagemanager.glade.h:87
msgid "Select packages by marking the checkbox and click to Install/Update."
msgstr "Выберите пакеты, отмечая флажок, и нажмите кнопку \"Установка/обновление\"."

#: ../gui/data/packagemanager.glade.h:88
msgid "Select packages by marking the checkbox and click to Remove selected."
msgstr "Выберите пакеты, отмечая флажок, и нажмите кнопку \"Удалить выбранное\"."

#: ../gui/data/packagemanager.glade.h:89
msgid "Sho_w:"
msgstr "Пока_зать"

#: ../gui/data/packagemanager.glade.h:90
msgid "Short description"
msgstr "Краткое описание"

#: ../gui/data/packagemanager.glade.h:91
msgid "Start Page"
msgstr "Начальная страница"

#: ../gui/data/packagemanager.glade.h:92
msgid "Start _Page"
msgstr "Начальная страница"

#: ../gui/data/packagemanager.glade.h:93
msgid "The folllowing will be added to your system"
msgstr "В систему будут добавлены следующие компоненты:"

#: ../gui/data/packagemanager.glade.h:94
msgid "This is a secure repository. Refer to the posted"
msgstr "Это хранилище является защищенным. Обратитесь к опубликованным сведениям"

#: ../gui/data/packagemanager.glade.h:95
msgid ""
"This repository requires a SSL Key and certificate.\n"
"Enter the location of these files in the spaces provided, or use the "
"Browse \n"
"buttons to provide their location."
msgstr ""
"Для доступа к данному хранилищу требуется ключ и сертификат SSL.\n"
"Введите местоположение этих файлов в приведенное поле или воспользуйтесь кнопками "
"Обзор \n"
"для ввода их местоположения."

#: ../gui/data/packagemanager.glade.h:98
msgid "Type text to search name and description of the package."
msgstr "Введите текст для поиска названия и описания пакета."

#: ../gui/data/packagemanager.glade.h:99 ../gui/modules/installupdate.py:222
#: ../packagemanager.py:368 ../packagemanager.py:3705
msgid "Update All"
msgstr "Обновить все"

#: ../gui/data/packagemanager.glade.h:100
msgid "Update all packages"
msgstr "Обновить все пакеты"

#: ../gui/data/packagemanager.glade.h:101 ../packagemanager.py:1360
msgid "Updates"
msgstr "Обновления"

#: ../gui/data/packagemanager.glade.h:102
msgid "_About"
msgstr "_О программе"

#: ../gui/data/packagemanager.glade.h:103
msgid "_Add"
msgstr "_Добавить"

#: ../gui/data/packagemanager.glade.h:104
msgid "_Browse"
msgstr "Про_смотреть"

#: ../gui/data/packagemanager.glade.h:105
msgid "_Clear"
msgstr "О_чистить"

#: ../gui/data/packagemanager.glade.h:106
msgid "_Contents"
msgstr "_Содержание"

#: ../gui/data/packagemanager.glade.h:107
msgid "_Copy"
msgstr "_Копировать"

#: ../gui/data/packagemanager.glade.h:108
msgid "_Deselect All"
msgstr "_О_тменить выбор всех"

#: ../gui/data/packagemanager.glade.h:109
msgid "_Edit"
msgstr "П_равка"

#: ../gui/data/packagemanager.glade.h:110
msgid "_Enable type ahead search (slow on large repositories)"
msgstr "Вкл_ючить автодополнение при вводе (низкая скорость в крупных хранилищах)"

#: ../gui/data/packagemanager.glade.h:111
msgid "_File"
msgstr "_Файл"

#: ../gui/data/packagemanager.glade.h:112
msgid "_Help"
msgstr "_Справка"

#: ../gui/data/packagemanager.glade.h:113
msgid "_Install / Update"
msgstr "_Установка/обновление"

#: ../gui/data/packagemanager.glade.h:114
msgid "_Instructions"
msgstr "_Инструкции"

#: ../gui/data/packagemanager.glade.h:115
msgid "_Manage Repositories..."
msgstr "У_правление хранилищами..."

#: ../gui/data/packagemanager.glade.h:116
msgid "_Modify..."
msgstr "_Изменить... "

#: ../gui/data/packagemanager.glade.h:117
msgid "_Name: "
msgstr "_Название: "

#: ../gui/data/packagemanager.glade.h:118
msgid "_Package"
msgstr "_Пакет"

#: ../gui/data/packagemanager.glade.h:119
msgid "_Paste"
msgstr "_Вставить"

#: ../gui/data/packagemanager.glade.h:120
msgid "_Proceed"
msgstr "_Продолжить"

#: ../gui/data/packagemanager.glade.h:121
msgid "_Quit"
msgstr "_Выйти"

#: ../gui/data/packagemanager.glade.h:122
msgid "_Remove"
msgstr "_Удалить"

#: ../gui/data/packagemanager.glade.h:123
msgid "_Search"
msgstr "_Поиск"

#: ../gui/data/packagemanager.glade.h:124
msgid "_Show Start Page on startup     "
msgstr "Вывод _начальной страницы при запуске     "

#: ../gui/data/packagemanager.glade.h:125
msgid "_URL: "
msgstr "_URL: "

#: ../gui/data/packagemanager.glade.h:126 ../um/data/updatemanager.glade.h:17
msgid "_Update All"
msgstr "_Обновить все"

#: ../gui/data/packagemanager.glade.h:127
msgid "_View"
msgstr "_Просмотр"

#: ../gui/data/packagemanager.glade.h:128 ../um/data/updatemanager.glade.h:18
msgid "gtk-cancel"
msgstr "gtk-cancel"

#: ../gui/data/packagemanager.glade.h:129 ../um/data/updatemanager.glade.h:19
msgid "gtk-close"
msgstr "gtk-close"

#: ../gui/data/packagemanager.glade.h:130 ../um/data/updatemanager.glade.h:20
msgid "gtk-help"
msgstr "gtk-help"

#: ../gui/data/packagemanager.glade.h:131
msgid "gtk-ok"
msgstr "gtk-ok"

#: ../gui/data/packagemanager.glade.h:132
msgid ""
"lf the repository requires a SSL key and certificate, or if it has mirrors, "
"add the new\n"
"repository, and then click Modify to add additional information."
msgstr ""
"Если для доступа к хранилищу требуется ключ и сертификат SSL, или если используются зеркала, "
"добавьте новое\n"
"хранилище и нажмите кнопку \"Изменить\" для добавления дополнительной информации."

#: ../gui/data/packagemanager-preferences.schemas.in.h:1
msgid ""
"A boolean which determines if the Start Page should be shown on startup or "
"the initial list view."
msgstr ""
"Булево значение, определяющее, что выводится при запуске - начальная страница или "
"начальный список."

#: ../gui/data/packagemanager-preferences.schemas.in.h:2
msgid ""
"A boolean which determines if the search is executed dynamically as user "
"types or only when user hits the return key."
msgstr ""
"Булево значение, определяющее необходимость динамического выполнения поиска при пользовательском вводе,"
"либо только после нажатия клавиши ввода."

#: ../gui/data/packagemanager-preferences.schemas.in.h:3
msgid ""
"A string which determines if the remote search repository is shown or not in "
"the error dialog."
msgstr ""
"Строка, указывающая необходимость вывода хранилища удаленного поиска "
"в диалоговом окне ошибки."

#: ../gui/data/packagemanager-preferences.schemas.in.h:4
msgid ""
"An integer which identifies the initial show filter value to be displayed "
"when packagemanager starts. 0 corresponds to All Packages."
msgstr ""
"Целочисленное значение, определяющее начальное значение фильтра отображения "
"при запуске диспетчера пакетов. 0 соответствует выводу всех пакетов."

#: ../gui/data/packagemanager-preferences.schemas.in.h:5
msgid ""
"An integer which identifies the initial toplevel value to be displayed when "
"packagemanager starts. 0 corresponds to All Categories."
msgstr ""
"Целочисленное значение, определяющее начальное значение верхнего уровня, отображаемого"
"при запуске диспетчера пакетов. 0 соответствует выводу всех категорий."

#: ../gui/data/packagemanager-preferences.schemas.in.h:6
msgid ""
"An integer which specifies the initial height of the application main "
"window, overridden if there is not enough screen available."
msgstr ""
"Целочисленное значение, указывающее начальную высоту главного "
"окна приложения; не используется при недостатке экранного пространства."

#: ../gui/data/packagemanager-preferences.schemas.in.h:7
msgid ""
"An integer which specifies the initial horizonatal divider position of the "
"application, which separates the category panel from the list panel."
msgstr ""
"Целочисленное значение, указывающее начальное положение горизонтального разделителя "
"приложения, отделяющего паниль категорий от панели списка."

#: ../gui/data/packagemanager-preferences.schemas.in.h:8
msgid ""
"An integer which specifies the initial vertical divider position of the "
"application, which separates the details panel from the list panel."
msgstr ""
"Целочисленное значение, указывающее начальное положение вертикального разделителя "
"приложения, отделяющего паниль подробных данных от панели списка."

#: ../gui/data/packagemanager-preferences.schemas.in.h:9
msgid ""
"An integer which specifies the initial width of the application main window, "
"overridden if there is not enough screen available."
msgstr ""
"Целочисленное значение, указывающее начальную ширину главного "
"окна приложения; не используется при недостатке экранного пространства."

#: ../gui/data/packagemanager-preferences.schemas.in.h:10
msgid ""
"Comma separated list of repositories which we do show if they doesn't "
"support remote search"
msgstr ""
"Разделенный запятыми список хранилищ, которые не должны выводиться, если в них не "
"реализована поддержка удаленного поиска"

#: ../gui/data/packagemanager-preferences.schemas.in.h:11
msgid "Enable type ahead search"
msgstr "Включение поиска с автодополнением при печати"

#: ../gui/data/packagemanager-preferences.schemas.in.h:12
msgid "Initial application height"
msgstr "Начальная высота приложения"

#: ../gui/data/packagemanager-preferences.schemas.in.h:13
msgid "Initial application horizonatal divider position"
msgstr "Начальное положение горизонтального разделителя"

#: ../gui/data/packagemanager-preferences.schemas.in.h:14
msgid "Initial application vertical divider position"
msgstr "Начальное положение вертикального разделителя"

#: ../gui/data/packagemanager-preferences.schemas.in.h:15
msgid "Initial application width"
msgstr "Начальная ширина приложения"

#: ../gui/data/packagemanager-preferences.schemas.in.h:16
msgid "Initial section value"
msgstr "Начальное значение раздела"

#: ../gui/data/packagemanager-preferences.schemas.in.h:17
msgid "Initial show filter value"
msgstr "Начальное значение фильтра отображения"

#: ../gui/data/packagemanager-preferences.schemas.in.h:18
msgid "Show Start Page on startup"
msgstr "Вывод начальной страницы при запуске"

#: ../gui/data/packagemanager-preferences.schemas.in.h:19
msgid "Start Page Update URL"
msgstr "Адрес URL обновления начальной страницы "

#: ../gui/data/packagemanager-preferences.schemas.in.h:20
msgid ""
"The location from which to fetch updates to the Package Manager Start Page."
msgstr ""
"Местоположение, из которого извлекаются обновления начальной страницы диспетчера пакетов."

#: ../gui/modules/webinstall.py:113
#, python-format
msgid "GUI will not respond to any event! %s. Check webinstall.py signals"
msgstr "Графический интерфейс не реагирует на события! %s. Проверьте сигналы webinstall.py"

#: ../gui/modules/webinstall.py:126
msgid ""
"\n"
" Add New Repository\n"
msgstr ""
"\n"
" Добавить новое хранилище\n"

#: ../gui/modules/webinstall.py:129
msgid ""
"\n"
" Add New Repositories\n"
msgstr ""
"\n"
" Добавить новые хранилища\n"

#: ../gui/modules/webinstall.py:138
msgid ""
"\n"
" All specified repositories and packages are already on the system.\n"
msgstr ""
"\n"
" Все указанные хранилища и пакеты уже имеются в системе.\n"

#: ../gui/modules/webinstall.py:147
#, python-format
msgid "\t%s "
msgstr "\t%s "

#: ../gui/modules/webinstall.py:151 ../packagemanager.py:769
#, python-format
msgid " (%s)\n"
msgstr " (%s)\n"

#: ../gui/modules/webinstall.py:156
msgid ""
"\n"
" Install Packages\n"
msgstr ""
"\n"
" Установить пакеты\n"

#: ../gui/modules/webinstall.py:164
#, python-format
msgid "\t%s: "
msgstr "\t%s: "

#: ../gui/modules/webinstall.py:167 ../gui/modules/installupdate.py:857
#: ../gui/modules/installupdate.py:907 ../updatemanager.py:1031
#, python-format
msgid "%s\n"
msgstr "%s\n"

#: ../gui/modules/webinstall.py:209
msgid "Proceed only if you trust these new repositories"
msgstr "Выполнение можно продолжать, только если новые хранилища заслуживают доверия"

#: ../gui/modules/webinstall.py:258 ../gui/modules/webinstall.py:386
#: ../gui/modules/webinstall.py:396
msgid "Repository Error"
msgstr "Ошибка хранилища"

#: ../gui/modules/webinstall.py:346
msgid "API Error"
msgstr "Ошибка интерфейса API"

#: ../gui/modules/webinstall.py:363
msgid "Error"
msgstr "Ошибка "

#: ../gui/modules/webinstall.py:393
msgid "Error reading the p5i file."
msgstr "Ошибка чтения файла p5i"

#: ../gui/modules/beadmin.py:73
msgid ""
"The <b>libbe</b> library was not found on your system.\n"
"All functions for managing Boot Environments are disabled"
msgstr ""
"Библиотека <b>libbe</b> в системе не найдена.\n"
"Все функции управления загрузочными средами отключены"

#: ../gui/modules/beadmin.py:81
msgid "BE management"
msgstr "Управление загрузочной средой "

#: ../gui/modules/beadmin.py:154
#, python-format
msgid "GUI will not respond to any event! %s. Check beadmin.py signals"
msgstr "Графический интерфейс не реагирует на события! %s. Проверьте сигналы beadmin.py"

#: ../gui/modules/beadmin.py:164
msgid "Loading Boot Environment Information"
msgstr "Загрузка информации о загрузочной среде"

#: ../gui/modules/beadmin.py:166
msgid "Fetching BE entries..."
msgstr "Получение записей загрузочной среды..."

#: ../gui/modules/beadmin.py:193
msgid "Boot Environment"
msgstr "Загрузочная среда"

#: ../gui/modules/beadmin.py:204
msgid "Created"
msgstr "Создано"

#: ../gui/modules/beadmin.py:213
msgid "Size"
msgstr "Размер"

#: ../gui/modules/beadmin.py:221
msgid "Active on Reboot"
msgstr "Активен после перезагрузки"

#: ../gui/modules/beadmin.py:232
msgid "Delete"
msgstr "Удалить"

#: ../gui/modules/beadmin.py:247 ../gui/modules/repository.py:470
#: ../gui/modules/repository.py:539 ../gui/modules/repository.py:727
#: ../gui/modules/repository.py:746 ../gui/modules/repository.py:867
msgid "Applying changes"
msgstr "Применение изменений"

#: ../gui/modules/beadmin.py:249 ../gui/modules/repository.py:471
#: ../gui/modules/repository.py:540 ../gui/modules/repository.py:728
#: ../gui/modules/repository.py:747 ../gui/modules/repository.py:868
msgid "Applying changes, please wait ..."
msgstr "Применение изменений, подождите..."

#: ../gui/modules/beadmin.py:278
msgid "Active on reboot:\n"
msgstr "Активен после перезапуска:\n"

#: ../gui/modules/beadmin.py:279
msgid "Delete boot environments:\n"
msgstr "Удалить среды загрузки:\n"

#: ../gui/modules/beadmin.py:280
msgid "Rename boot environments:\n"
msgstr "Переименовать среды загрузки:\n"

#: ../gui/modules/beadmin.py:319
msgid " to "
msgstr " в "

#: ../gui/modules/beadmin.py:373
msgid "<b>Couldn't change Active Boot Environment to:</b>\n"
msgstr "<b>Не удалось изменить активную загрузочную среду на:</b>\n"

#: ../gui/modules/beadmin.py:378
msgid "<b>Couldn't delete Boot Environments:</b>\n"
msgstr "<b>Не удалось удалить загрузочные среды:</b>\n"

#: ../gui/modules/beadmin.py:385
msgid "<b>Couldn't rename Boot Environments:</b>\n"
msgstr "<b>Не удалось переименовать среды загрузки:</b>\n"

#: ../gui/modules/beadmin.py:388
#, python-format
msgid "%s <b>to</b> %s\n"
msgstr "%s <b>в</b> %s\n"

#: ../gui/modules/beadmin.py:407
msgid "BE error"
msgstr "Ошибка среды загрузки"

#: ../gui/modules/beadmin.py:492
msgid ""
"The <b>libbe</b> library couldn't  prepare list of Boot Environments.\n"
"All functions for managing Boot Environments are disabled"
msgstr ""
"Библиотека <b>libbe</b> не может подготовить список загрузочных сред.\n"
"Все функции управления загрузочными средами отключены"

#: ../gui/modules/beadmin.py:522 ../gui/modules/beadmin.py:538
msgid "%m/%d/%y %H:%M"
msgstr "%m/%d/%y %H:%M"

#: ../gui/modules/installupdate.py:100
msgid "Preparing..."
msgstr "Подготовка..."

#: ../gui/modules/installupdate.py:101 ../gui/modules/installupdate.py:544
msgid "Downloading..."
msgstr "Загрузка..."

#: ../gui/modules/installupdate.py:102
msgid "Installing..."
msgstr "Установка..."

#: ../gui/modules/installupdate.py:191
#, python-format
msgid "GUI will not respond to any event! %s. Check installupdate.py signals"
msgstr "Графический интерфейс не реагирует на события! %s. Проверьте сигналы installupdate.py"

#. We are not showing the download stage in the main stage list
#: ../gui/modules/installupdate.py:203
msgid "Removing..."
msgstr "Удаление..."

#: ../gui/modules/installupdate.py:252
msgid "Canceling..."
msgstr "Отмена..."

#: ../gui/modules/installupdate.py:356
msgid "BE name is in use"
msgstr "Имя среды загрузки уже используется"

#: ../gui/modules/installupdate.py:358
msgid "BE name is invalid"
msgstr "Недопустимое имя среды загрузки "

#: ../gui/modules/installupdate.py:392
#, python-format
msgid "Ensuring %s is up to date..."
msgstr "Проверка актуальности %s..."

#: ../gui/modules/installupdate.py:413
msgid ""
"Accessing this restricted repository failed.\n"
"You either need to register to access this repository,\n"
"the certificate expired, or you need to accept the repository\n"
"certificate."
msgstr ""
"Не удалось получить доступ к хранилищу с ограничением доступа.\n"
"Для доступа к хранилищу необходимо зарегистрироваться,\n"
"срок действия сертификата истек или его необходимо\n"
"принять."

#: ../gui/modules/installupdate.py:423
msgid "Inventory exception:\n"
msgstr "Исключительная ситуация инвентаризационной информации:\n"

#: ../gui/modules/installupdate.py:430
msgid ""
"Please check the network connection.\n"
"Is the repository accessible?"
msgstr ""
"Проверьте соединение с Интернетом.\n"
"Хранилище доступно?"

#: ../gui/modules/installupdate.py:440
#, python-format
msgid ""
"Please check the network connection.\n"
"Is the repository accessible?\n"
"\n"
"%s"
msgstr ""
"Проверьте сетевое соединение.\n"
"Возможен ли доступ к хранилищу?\n"
"\n"
"%s"

#: ../gui/modules/installupdate.py:446
msgid ""
"pkg(5) appears to be out of date and should be\n"
"updated before running Update All.\n"
"Please update SUNWipkg package"
msgstr ""
"pkg(5) оказался устаревшим и должен быть обновлен\n"
" до выполнения действия \"Обновить все\".\n"
"Обновите пакет SUNWipkg"

#: ../gui/modules/installupdate.py:452
#, python-format
msgid ""
"Cannot remove:\n"
"\t%s\n"
"Due to the following packages that depend on it:\n"
msgstr ""
"Невозможно удалить:\n"
"\t%s\n"
"вследствие зависимости следующих пакетов:\n"

#: ../gui/modules/installupdate.py:461
#, python-format
msgid ""
"\n"
"Failure of consistent use of pfexec or gksu when running\n"
"%s is often a source of this problem."
msgstr ""
"\n"
"Непоследовательное использование pfexec или gksu при запуске\n"
"Источником этой проблемы часто бывает %s."

#: ../gui/modules/installupdate.py:464
msgid ""
"\n"
"To rebuild index, please use the terminal command:"
msgstr ""
"\n"
"Для перестройки индекса используется команда терминала:"

#: ../gui/modules/installupdate.py:465
msgid ""
"\n"
"\tpfexec pkg rebuild-index"
msgstr ""
"\n"
"\tpfexec pkg rebuild-index"

#: ../gui/modules/installupdate.py:469
msgid ""
"There was an error during installation. The search\n"
"index is corrupted. You might want try to fix this\n"
"problem by running command:\n"
"\tpfexec pkg rebuild-index"
msgstr ""
"Во время установки произошла ошибка.\n"
"Индекс поврежден. Можно попробовать исправить\n"
"данную проблему с помощью команды:\n"
"\tpfexec pkg rebuild-index"

#: ../gui/modules/installupdate.py:476
msgid ""
"This is an Live Image. The install\n"
"operation can't be performed."
msgstr ""
"Это образ Live Image. Установка\n"
"не может быть выполнена."

#: ../gui/modules/installupdate.py:481
#, python-format
msgid ""
"There was an error during installation.\n"
"The Plan of the operation is missing and the operation\n"
"can't be finished. You might want try to fix this\n"
"problem by restarting %s\n"
msgstr ""
"Во время установки произошла ошибка.\n"
"Отсутствует план действия, и действие \n"
"не может быть выполнено. Для устранения этой проблемы может\n"
"потребоваться перезапуск %s\n"

#: ../gui/modules/installupdate.py:488
#, python-format
msgid ""
"There was an error during installation.\n"
"The State of the image is incorrect and the operation\n"
"can't be finished. You might want try to fix this\n"
"problem by restarting %s\n"
msgstr ""
"Во время установки произошла ошибка.\n"
"Состояние образа является некорректным, и действие\n"
"не может быть выполнено. Для устранения этой проблемы может\n"
"потребоваться перезапуск %s\n"

#: ../gui/modules/installupdate.py:499
msgid "Specifying BE Name not supported.\n"
msgstr "Указанное имя среды загрузки не поддерживается.\n"

#: ../gui/modules/installupdate.py:503
#, python-format
msgid "Invalid BE Name: %s.\n"
msgstr "Недопустимое имя среды загрузки: %s.\n"

#: ../gui/modules/installupdate.py:540
#, python-format
msgid "Updating %s"
msgstr "Обновление %s"

#: ../gui/modules/installupdate.py:546
msgid "Executing..."
msgstr "Выполнение..."

#: ../gui/modules/installupdate.py:553
msgid "Gathering packages information, please wait..."
msgstr "Сбор информации о пакетах, ждите..."

#: ../gui/modules/installupdate.py:575
msgid "All packages already installed."
msgstr "Все пакеты уже установлены."

#: ../gui/modules/installupdate.py:580
msgid ""
"Selected package(s) cannot be updated on their own.\n"
"Click Update All to update all packages."
msgstr ""
"Выбранный пакет (пакеты) невозможно обновить независимо.\n"
"Нажмите \"Update All\" для обновления всех пакетов."

#: ../gui/modules/installupdate.py:583
msgid "Your system has already been updated."
msgstr "Система уже обновлена."

#: ../gui/modules/installupdate.py:621 ../gui/modules/installupdate.py:656
msgid "No futher information available"
msgstr "Дополнительная информация отсутствует."

#: ../gui/modules/installupdate.py:624 ../gui/modules/installupdate.py:647
msgid ""
"\n"
"Error:\n"
msgstr ""
"\n"
"Ошибка:\n"

#: ../gui/modules/installupdate.py:627 ../gui/modules/installupdate.py:637
msgid " - Failed </b>"
msgstr " - отказ </b>"

#: ../gui/modules/installupdate.py:641
#, python-format
msgid ""
"An unknown error occured in the %s stage.\n"
"Please let the developers know about this problem\n"
"by filing a bug together with exception value at:\n"
msgstr ""
"На этапе %s возникла неизвестная ошибка.\n"
"Сообщите разработчикам об этой проблеме\n"
"путем подачи заявления об ошибке со значением исключения по адресу:\n"

#: ../gui/modules/installupdate.py:645
msgid ""
"http://defect.opensolaris.org\n"
"\n"
msgstr ""
"http://defect.opensolaris.org\n"
"\n"

#: ../gui/modules/installupdate.py:646
msgid "Exception value:\n"
msgstr "Значение исключения:\n"

#: ../gui/modules/installupdate.py:651
msgid "Exception traceback:\n"
msgstr "Результат отслеживания исключениия:\n"

#: ../gui/modules/installupdate.py:708
#, python-format
msgid "Downloaded %(current)s of %(total)s"
msgstr "Загружено %(current)s of %(total)s"

#: ../gui/modules/installupdate.py:728
msgid "Catalog refresh failed during install."
msgstr "Ошибка обновления каталога при установке."

#: ../gui/modules/installupdate.py:743
msgid "Catalog refresh failed during Update All."
msgstr "Ошибка обновления каталога при обновлении всех пакетов."

#: ../gui/modules/installupdate.py:747
msgid "Installation completed successfully."
msgstr "Установка успешно завершена."

#: ../gui/modules/installupdate.py:749
msgid "Packages removed successfully."
msgstr "Удаление пакетов успешно завершено."

#: ../gui/modules/installupdate.py:751
msgid "Packages updated successfully."
msgstr "Обновление пакетов успешно завершено."

#: ../gui/modules/installupdate.py:782
msgid ""
"Not enough disk space, the selected action cannot be performed.\n"
"\n"
"Click OK to manage your existing BEs and free up disk space or Cancel to "
"cancel the action."
msgstr ""
"Недостаточно места на диске, выполнение выбранного действия невозможно.\n"
"\n"
"Нажмите кнопку \"OK\" для управления имеющимися средами загрузки в целях освобождения дискового пространства либо нажмите кнопку \"Отмена\" для отмены "
"действия."

#: ../gui/modules/installupdate.py:786
msgid "Not Enough Disk Space"
msgstr "Недостаточно места на диске"

#: ../gui/modules/installupdate.py:804
msgid "Packages To Be Updated:\n"
msgstr "Обновляемые пакетоы: \n"

#: ../gui/modules/installupdate.py:811
msgid "Packages To Be Installed:\n"
msgstr "Устанавливаемые пакеты:\n"

#: ../gui/modules/installupdate.py:818
msgid "Packages To Be Removed:\n"
msgstr "Удаляемые пакеты:\n"

#: ../gui/modules/installupdate.py:877
#, python-format
msgid "Package %d of %d: %s\n"
msgstr "Пакет %d из %d: %s\n"

#: ../gui/modules/installupdate.py:896
#, python-format
msgid "Evaluating: %s"
msgstr "Анализ: %s"

#: ../gui/modules/repository.py:266
#, python-format
msgid "GUI will not respond to any event! %s. Check repository.py signals"
msgstr "Графический интерфейс не реагирует на события! %s. Проверьте сигналы repository.py"

#: ../gui/modules/repository.py:309
msgid "Add New Repository"
msgstr "Добавить новое хранилище"

#: ../gui/modules/repository.py:317
msgid "<b>New Repository</b>"
msgstr "<b>Новое хранилище</b>"

#: ../gui/modules/repository.py:363
msgid "Repository Name"
msgstr "Название хранилища"

#: ../gui/modules/repository.py:372
msgid "Preferred"
msgstr "Предпочтительно"

#: ../gui/modules/repository.py:384
msgid "Enabled"
msgstr "Включено"

#: ../gui/modules/repository.py:505
#, python-format
msgid "Failed to disable %s.\n"
msgstr "Отключение %s не удалось.\n"

#: ../gui/modules/repository.py:507
#, python-format
msgid "Failed to enable %s.\n"
msgstr "Включение %s не удалось.\n"

#: ../gui/modules/repository.py:513 ../gui/modules/repository.py:521
#: ../gui/modules/repository.py:531
#, python-format
msgid "Failed to disable %s."
msgstr "Отключение %s не удалось."

#: ../gui/modules/repository.py:515 ../gui/modules/repository.py:523
#: ../gui/modules/repository.py:533
#, python-format
msgid "Failed to enable %s."
msgstr "Включение %s не удалось."

#: ../gui/modules/repository.py:516 ../gui/modules/repository.py:985
#: ../gui/modules/repository.py:1013
msgid ""
"\n"
"Please check your permissions."
msgstr ""
"\n"
"Проверьте свои полномочия."

#: ../gui/modules/repository.py:525 ../gui/modules/repository.py:995
msgid ""
"\n"
"Please check the network connection or URL.\n"
"Is the repository accessible?"
msgstr ""
"\n"
"Проверьте соединение с Интернетом или URL.\n"
"Хранилище доступно?"

#: ../gui/modules/repository.py:534
msgid ""
"\n"
"Unexpected error.\n"
msgstr ""
"\n"
"Непредвиденная ошибка.\n"

#: ../gui/modules/repository.py:577
msgid ""
"Couldn't change the preferred publisher.\n"
"Please check your permissions."
msgstr ""
"Не удалось изменить предпочтительного издателя.\n"
"Проверьте свои полномочия."

#: ../gui/modules/repository.py:641
msgid "Name contains invalid characters"
msgstr "Имя содержит недопустимые символы"

#: ../gui/modules/repository.py:648
msgid "Name already in use"
msgstr "Имя уже используется"

#: ../gui/modules/repository.py:669
msgid "URL is not valid"
msgstr "Недействительный адрес URL"

#: ../gui/modules/repository.py:855 ../packagemanager.py:2217
#: ../updatemanager.py:926
#, python-format
msgid ""
"Unable to navigate to:\n"
"\t%s"
msgstr ""
"Невозможен переход к :\n"
"\t%s"

#: ../gui/modules/repository.py:856
msgid "Registration"
msgstr "Регистрация"

#: ../gui/modules/repository.py:863
msgid "Adding New Repository"
msgstr "Добавление нового хранилища"

#: ../gui/modules/repository.py:864
#, python-format
msgid ""
"Adding:\n"
"\t%s (%s)..."
msgstr ""
"Добавление:\n"
"\t%s (%s)..."

#: ../gui/modules/repository.py:969
#, python-format
msgid ""
"Failed to add repository: %s\n"
"\n"
msgstr ""
"Не удалось добавить хранилище: %s\n"
"\n"

#: ../gui/modules/repository.py:976
#, python-format
msgid "Failed to add %s.\n"
msgstr "Добавление %s не удалось.\n"

#: ../gui/modules/repository.py:984 ../gui/modules/repository.py:993
#, python-format
msgid "Failed to add %s."
msgstr "Не удалось добавить %s."

#: ../gui/modules/repository.py:1012
#, python-format
msgid "Failed to delete %s."
msgstr "Не удалось удалить %s."

#: ../gui/modules/repository.py:1120 ../gui/modules/repository.py:1155
msgid "Specify SSL Key File"
msgstr "Укажите файл ключей SSL"

#: ../gui/modules/repository.py:1140 ../gui/modules/repository.py:1175
msgid "Specify SSL Certificate File"
msgstr "Укажите файл сертификатов SSL"

#. More Generic for WebInstall
#: ../gui/modules/repository.py:1212
msgid "Repository error"
msgstr "Ошибка хранилища"

#: ../modules/actions/__init__.py:89
#, python-format
msgid ""
"unknown action type '%(type)s' in package '%(fmri)s' in action '%(action)s'"
msgstr ""
"неизвестный тип действия '%(type)s' в пакете '%(fmri)s' в действии '%(action)s'"

#: ../modules/actions/__init__.py:93
#, python-format
msgid "unknown action type '%(type)s' in action '%(action)s'"
msgstr "неизвестный тип действия '%(type)s' в действии '%(action)s'"

#: ../modules/actions/__init__.py:106
#, python-format
msgid ""
"Malformed action in package '%(fmri)s' at position: %(pos)d:\n"
"    %(action)s\n"
"%(marker)s"
msgstr ""
"Недопустимое действие в пакете '%(fmri)s' в позиции: %(pos)d:\n"
"    %(action)s\n"
"%(marker)s"

#: ../modules/actions/__init__.py:111
#, python-format
msgid ""
"Malformed action at position: %(pos)d:\n"
"    %(action)s\n"
"%(marker)s"
msgstr ""
"Недопустимое действие в позиции: %(pos)d:\n"
"    %(action)s\n"
"%(marker)s"

#: ../modules/actions/__init__.py:137
#, python-format
msgid "invalid action in package %(fmri)s: %(action)s: %(error)s"
msgstr "недопустимое действие в пакете %(fmri)s: %(action)s: %(error)s"

#: ../modules/actions/__init__.py:140
#, python-format
msgid "invalid action, '%(action)s': %(error)s"
msgstr "недопустимое действие, '%(action)s': %(error)s"

#: ../modules/actions/__init__.py:164
#, python-format
msgid "required attribute '%s' was not provided."
msgstr "не предоставлен обязательный атрибут '%s'."

#: ../modules/actions/__init__.py:221
#, python-format
msgid "required attribute, '%s', was not provided."
msgstr "не предоставлен обязательный атрибут '%s'."

#: ../modules/client/api_errors.py:112
#, python-format
msgid ""
"Could not operate on %s\n"
"because of insufficient permissions. Please try the command again using "
"pfexec\n"
"or otherwise increase your privileges."
msgstr ""
"Работа с %s невозможна\n"
"из-за недостаточных полномочий. Повторите попытку вызова команды с помощью"
"pfexec\n"
"или повысьте уровень полномочий другим способом."

#: ../modules/client/api_errors.py:117
msgid ""
"\n"
"Could not complete the operation because of insufficient permissions. "
"Please\n"
"try the command again using pfexec or otherwise increase your privileges.\n"
msgstr ""
"\n"
"Выполнение действия невозможна из-за недостаточных полномочий. "
"Повторите\n"
"попытку вызова команды с помощью pfexec или повысьте уровень полномочий другим способом.\n"

#: ../modules/client/api_errors.py:128
#, python-format
msgid ""
"Could not operate on %s\n"
"because the file is in use. Please stop using the file and try the\n"
"operation again."
msgstr ""
"Работа с %s невозможна,\n"
"поскольку файл используется. Прекратите использование этого файла и повторите попытку\n"
"выполнения действия."

#: ../modules/client/api_errors.py:147
msgid ""
"The following pattern(s) did not match any packages in the current catalog.\n"
"Try relaxing the pattern, refreshing and/or examining the catalogs:"
msgstr ""
"Следующие шаблоны не подходят ни к одному из пакетов в текущем\n"
"каталоге. Примените менее узкие критерии поиска, обновите и/или просмотрите каталоги:"

#: ../modules/client/api_errors.py:160
#, python-format
msgid "'%s' matches no installed packages"
msgstr "'%s' не соответствует ни одному из установленных пакетов"

#: ../modules/client/api_errors.py:163
#, python-format
msgid "'%s' is an illegal fmri"
msgstr "'%s' является недопустимым fmri"

#: ../modules/client/api_errors.py:167
msgid "The following package(s) violated constraints:"
msgstr "Нарушение ограничений для следующих пакетов:"

#: ../modules/client/api_errors.py:172
#, python-format
msgid "'%s' supports the following architectures: %s"
msgstr "'%s' поддерживает следующие архитектуры: %s"

#: ../modules/client/api_errors.py:173
#, python-format
msgid "Image architecture is defined as: %s"
msgstr "Архитектура образа определена как: %s"

#: ../modules/client/api_errors.py:239
msgid ""
"No matching package could be found for the following FMRIs in any of the "
"catalogs for the current publishers:\n"
msgstr ""
"Не удалось найти ни один соответствующий пакет для следующих FMRI "
"ни в одном из каталогов для текущих издателей:\n"

#: ../modules/client/api_errors.py:337
#, python-format
msgid ""
"The specified file is in an unrecognized format or does not contain valid "
"publisher information: %s"
msgstr ""
"Указанный файл имеет неопознанный формат, либо не содержит действительных"
"данных об издателе: %s"

#: ../modules/client/api_errors.py:340
msgid ""
"The specified file is in an unrecognized format or does not contain valid "
"publisher information."
msgstr ""
"Указанный файл имеет неопознанный формат, либо не содержит действительных"
"данных об издателе."

#: ../modules/client/api_errors.py:349
msgid "Unsupported pkg(5) publisher information data format."
msgstr "Неподдерживаемый формат данных об издателе pkg(5)."

#: ../modules/client/api_errors.py:375
#, python-format
msgid ""
"Error encountered while retrieving data from '%s':\n"
"%s"
msgstr ""
"Ошибка при извлечении данных из '%s':\n"
"%s"

#: ../modules/client/api_errors.py:377
#, python-format
msgid "Error encountered while retrieving data from: %s"
msgstr "Ошибка при извлечении данных из: %s"

#: ../modules/client/api_errors.py:385
#, python-format
msgid "'%s' is not a valid location."
msgstr "'%s' не является допустимым местоположением."

#: ../modules/client/api_errors.py:397
#, python-format
msgid "'%s' is not a valid boot envirnment name."
msgstr "'%s' не является действительным именем среды загрузки."

#: ../modules/client/api_errors.py:406
msgid ""
"Boot environment naming during package install is not supported on this\n"
"version of OpenSolaris. Please image-update without the --be-name option."
msgstr ""
"Присвоение названий средам загрузки при установке пакетов не поддерживается в данной\n"
"версии OpenSolaris. Выполните image-update без параметра --be-name."

#: ../modules/client/api_errors.py:412
msgid "Unable to clone the current boot environment."
msgstr "Невозможно клонировать текущую среду загрузки."

#: ../modules/client/api_errors.py:425
#, python-format
msgid ""
"A problem occurred while attempting to rename the boot environment\n"
"currently named %(orig)s to %(dest)s."
msgstr ""
"Проблема при попытке переименования среды загрузки \n"
"с текущим именем %(orig)s в %(dest)s."

#: ../modules/client/api_errors.py:436
#, python-format
msgid "Unable to mount %(name)s at %(mt)s"
msgstr "Не удалось смонтировать %(name)s на %(mt)s"

#: ../modules/client/api_errors.py:445
msgid ""
"Naming a boot environment when operating on a non-live image is\n"
"not allowed."
msgstr ""
"Присвоение имени среде загрузки при работе с неактивным образом\n"
"не допускается."

#: ../modules/client/api_errors.py:456
msgid "Info does not recognize the following options:"
msgstr "Команда info не распознает следующие параметры:"

#: ../modules/client/api_errors.py:458
msgid " '"
msgstr " '"

#: ../modules/client/api_errors.py:458
msgid "'"
msgstr "'"

#: ../modules/client/api_errors.py:468
msgid "Some servers failed to respond appropriately:\n"
msgstr "От некоторых серверов не получен соответствующий ответ:\n"

#: ../modules/client/api_errors.py:473
#, python-format
msgid "    %(o)s: %(msg)s (%(code)d)\n"
msgstr "    %(o)s: %(msg)s (%(code)d)\n"

#: ../modules/client/api_errors.py:478
#, python-format
msgid "    %s: timeout\n"
msgstr "    %s: истекло время ожидания\n"

#: ../modules/client/api_errors.py:481
#, python-format
msgid "    %(o)s: %(other)s\n"
msgstr "    %(o)s: %(other)s\n"

#: ../modules/client/api_errors.py:485
#, python-format
msgid ""
"    %(o)s: Unable to read status of HTTP response:%(l)s\n"
"        This is most likely not a pkg(5) depot.  Please check the URL and "
"the \n"
"        port number."
msgstr ""
"    %(o)s: невозможно считать статус ответа HTTP:%(l)s\n"
"        Вероятно, указанный адрес не соответствует депо pkg(5).  Проверьте адрес URL и "
"номер \n"
"        порта."

#: ../modules/client/api_errors.py:493
#, python-format
msgid "    %s: Incomplete read from host"
msgstr "    %s: неполное чтение с узла"

#: ../modules/client/api_errors.py:496
#, python-format
msgid "    %(o)s: %(msg)s\n"
msgstr "    %(o)s: %(msg)s\n"

#: ../modules/client/api_errors.py:499
#, python-format
msgid "    %s: Socket timeout"
msgstr "    %s: истекло время ожидания сокета"

#: ../modules/client/api_errors.py:501
#, python-format
msgid "    %(o)s: Socket error, reason: %(msg)s"
msgstr "    %(o)s: ошибка сокета, причина: %(msg)s"

#: ../modules/client/api_errors.py:505
#, python-format
msgid "    %(o)s: %(msg)s"
msgstr "    %(o)s: %(msg)s"

#: ../modules/client/api_errors.py:508
#, python-format
msgid "%s appears not to be a valid package depot.\n"
msgstr "%s не является допустимым хранилищем пакетов.\n"

#: ../modules/client/api_errors.py:531
msgid ""
"Search capabilities and performance are degraded.\n"
"To improve, run 'pkg rebuild-index'."
msgstr ""
"Возможности и скорость поиска снижены.\n"
"Для снятия ограничений запустите команду 'pkg rebuild-index'."

#: ../modules/client/api_errors.py:571
#, python-format
msgid "'%s' is not a valid publisher name."
msgstr "'%s' не является допустимым именем издателя."

#: ../modules/client/api_errors.py:579
#, python-format
msgid ""
"'%(value)s' is not a valid value for repository attribute '%(attribute)s'."
msgstr ""
"'%(value)s' не является допустимым значением параметра хранилища '%(attribute)s'."

#: ../modules/client/api_errors.py:592
#, python-format
msgid "'%s' is not a valid repository collection type."
msgstr "'%s' не является допустимым типом набора хранилища."

#: ../modules/client/api_errors.py:600
#, python-format
msgid "'%s' is not a valid URI."
msgstr "'%s' не является допустимым идентификатором URI."

#: ../modules/client/api_errors.py:608
#, python-format
msgid "'%s' is not a valid URI priority; integer value expected."
msgstr "'%s' не является допустимым значением приоритета URI; ожидается целочисленное значение."

#: ../modules/client/api_errors.py:620
#, python-format
msgid "'%s' is not a valid repository URI sort policy."
msgstr "'%s' не является допустимой политикой сортировки URI хранилищ."

#: ../modules/client/api_errors.py:629
#, python-format
msgid "Publisher '%s' is disabled and cannot be used for packaging operations."
msgstr "Издатель'%s' отключен, и не может использоваться для работы с пакетами."

#: ../modules/client/api_errors.py:638
#, python-format
msgid "A publisher with the same name or alias as '%s' already exists."
msgstr "Издатель с таким же именем или псевдонимом, что и '%s', уже существует."

#: ../modules/client/api_errors.py:647
#, python-format
msgid ""
"A repository with the same name or origin URIs already exists for publisher "
"'%s'."
msgstr ""
"Хранилище с таким же именем или идентификаторами URI источников уже существует для издателя "
"'%s'."

#: ../modules/client/api_errors.py:656
#, python-format
msgid "Mirror '%s' already exists for the specified repository."
msgstr "Для указанного хранилища уже существует зеркало '%s'."

#: ../modules/client/api_errors.py:665
#, python-format
msgid "Origin '%s' already exists for the specified repository."
msgstr "Для указанного хранилища уже существует источник '%s'."

#: ../modules/client/api_errors.py:674
msgid "The preferred publisher cannot be removed."
msgstr "Удаление предпочтительного издателя невозможно."

#: ../modules/client/api_errors.py:682
msgid "Cannot remove the selected repository for a publisher."
msgstr "Удаление выбранного хранилища для издателя невозможно."

#: ../modules/client/api_errors.py:691
#, python-format
msgid ""
"Publisher '%s' is disabled and cannot be set as the preferred publisher."
msgstr ""
"Издатель '%s' отключен, и его невозможно выбрать в качестве предпочтительного издателя."

#: ../modules/client/api_errors.py:700
#, python-format
msgid "Unknown legal URI '%s'."
msgstr "Неизвестный допустимый идентификатор URI '%s'."

#: ../modules/client/api_errors.py:708
#, python-format
msgid "Unknown publisher '%s'."
msgstr "Неизвестный издатель '%s'."

#: ../modules/client/api_errors.py:716
#, python-format
msgid "Unknown related URI '%s'."
msgstr "Неизвестный связанный идентификатор URI '%s'."

#: ../modules/client/api_errors.py:724
#, python-format
msgid "Unknown repository '%s'."
msgstr "Неизвестное хранилище '%s'."

#: ../modules/client/api_errors.py:732
#, python-format
msgid "Unknown repository mirror '%s'."
msgstr "Неизвестное зеркало хранилища '%s'."

#: ../modules/client/api_errors.py:740
#, python-format
msgid "Unknown repository origin '%s'"
msgstr "Неизвестный источник хранилища '%s'"

#: ../modules/client/api_errors.py:751
#, python-format
msgid "The URI '%(uri)s' contains an unsupported scheme '%(scheme)s'."
msgstr "Идентификатор URI '%(uri)s' содержит неподдерживаемую схему '%(scheme)s'."

#: ../modules/client/api_errors.py:754
msgid "The specified URI contains an unsupported scheme."
msgstr "Указанный идентификатор URI содержит неподдерживаемую схему."

#: ../modules/client/api_errors.py:762
#, python-format
msgid "'%(attr)s' is not supported for '%(scheme)s'."
msgstr "'%(attr)s' не поддерживается для '%(scheme)s'."

#: ../modules/client/api_errors.py:789
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s' needed to access '%(uri)s', "
"has expired.  Please install a valid certificate."
msgstr ""
"Срок действия сертификата '%(cert)s' для издателя '%(pub)s, требуемого для доступа к '%(uri)s', "
"истек.  Установите действующий сертификат."

#: ../modules/client/api_errors.py:794
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', has expired.  Please install "
"a valid certificate."
msgstr ""
"Срок действия сертификата '%(cert)s' для издателя '%(pub)s' истек.  Установите"
"действующий сертификат."

#: ../modules/client/api_errors.py:799
#, python-format
msgid ""
"Certificate '%(cert)s', needed to access '%(uri)s', has expired.  Please "
"install a valid certificate."
msgstr ""
"Срок действия сертификата '%(cert)s', требуемого для доступа к '%(uri)s', истек. Установите"
"действующий сертификат."

#: ../modules/client/api_errors.py:802
#, python-format
msgid "Certificate '%s' has expired.  Please install a valid certificate."
msgstr "Срок действия сертификата '%s' истек. Установите действующий сертификат."

#: ../modules/client/api_errors.py:815
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', needed to access '%(uri)s', "
"will expire in '%(days)s' days."
msgstr ""
"Срок действия сертификата '%(cert)s' для издателя '%(pub)s', требуемого для доступа к '%(uri)s', "
"истечет через '%(days)s' дней."

#: ../modules/client/api_errors.py:820
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s' will expire in '%(days)s' "
"days."
msgstr ""
"Срок действия сертификата '%(cert)s' для издателя '%(pub)s' истечет через '%(days)s' "
"дней."

#: ../modules/client/api_errors.py:824
#, python-format
msgid ""
"Certificate '%(cert)s', needed to access '%(uri)s', will expire in '%(days)"
"s' days."
msgstr ""
"Срок действия сертификата '%(cert)s', требуемого для доступа к '%(uri)s', истечет через '%(days)"
"s' дней."

#: ../modules/client/api_errors.py:827
#, python-format
msgid "Certificate '%(cert)s' will expire in '%(days)s' days."
msgstr "Срок действия сертификата '%(cert)s' истечет через '%(days)s' дней."

#: ../modules/client/api_errors.py:839
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', needed to access '%(uri)s', "
"is invalid."
msgstr ""
"Сертификат '%(cert)s' для издателя '%(pub)s', требуемый для доступа к '%(uri)s', "
"является недействительным."

#: ../modules/client/api_errors.py:843
#, python-format
msgid "Certificate '%(cert)s' for publisher '%(pub)s' is invalid."
msgstr "Сертификат '%(cert)s' для издателя '%(pub)s' является недействительным."

#: ../modules/client/api_errors.py:847
#, python-format
msgid "Certificate '%(cert)s' needed to access '%(uri)s' is invalid."
msgstr "Сертификат '%(cert)s', требуемый для доступа к '%(uri)s', является недействительным."

#: ../modules/client/api_errors.py:850
#, python-format
msgid "Invalid certificate '%s'."
msgstr "Недействительный сертификат '%s'."

#: ../modules/client/api_errors.py:861
#, python-format
msgid ""
"Unable to locate certificate '%(cert)s' for publisher '%(pub)s' needed to "
"access '%(uri)s'."
msgstr ""
"Не обнаружен сертификат '%(cert)s' для издателя '%(pub)s', требуемый для "
"доступа к '%(uri)s'."

#: ../modules/client/api_errors.py:866
#, python-format
msgid "Unable to locate certificate '%(cert)s' for publisher '%(pub)s'."
msgstr "Не обнаружен сертификат '%(cert)s' для издателя '%(pub)s'."

#: ../modules/client/api_errors.py:870
#, python-format
msgid "Unable to locate certificate '%(cert)s' needed to access '%(uri)s'."
msgstr "Не обнаружен сертификат '%(cert)s', требуемый для доступа к '%(uri)s'."

#: ../modules/client/api_errors.py:873
#, python-format
msgid "Unable to locate certificate '%s'."
msgstr "Не обнаружен сертификат '%s'."

#: ../modules/client/api_errors.py:885
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s', needed to access '%(uri)s', "
"has a future effective date."
msgstr ""
"Дата вступления в силу сертификата '%(cert)s' для издателя '%(pub)s', требуемого для доступа к '%(uri)s', "
"находится в будущем."

#: ../modules/client/api_errors.py:890
#, python-format
msgid ""
"Certificate '%(cert)s' for publisher '%(pub)s' has a future effective date."
msgstr ""
"Дата вступления в силу сертификата '%(cert)s' для издателя '%(pub)s' находится в будущем."

#: ../modules/client/api_errors.py:894
#, python-format
msgid ""
"Certificate '%(cert)s' needed to access '%(uri)s' has a future effective "
"date."
msgstr ""
"Дата вступления в силу сертификата '%(cert)s', требуемого для доступа к '%(uri)s', находится в "
"будущем."

#: ../modules/client/api_errors.py:897
#, python-format
msgid "Certificate '%s' has a future effective date."
msgstr "Дата вступления в силу сертификата '%s' находится в будущем."

#: ../modules/client/api_errors.py:907
#, python-format
msgid "Gave a bad response:%s"
msgstr "Некорректный ответ:%s"

#: ../modules/client/bootenv.py:132
msgid "pkg: unable to create an auto snapshot. pkg recovery is disabled."
msgstr "pkg: невозможно создать автоснимок. Восстановление pkg  запрещено."

#: ../modules/client/bootenv.py:249
#, python-format
msgid "pkg: A system error %s was caught executing %s"
msgstr "pkg: Произошла системная ошибка %s при выполнении %s"

#: ../modules/client/bootenv.py:253
#, python-format
msgid ""
"pkg: '%s' failed. \n"
"with a return code of %d."
msgstr ""
"pkg: Сбой '%s' \n"
"с кодом завершения %d."

#: ../modules/client/bootenv.py:264
#, python-format
msgid "pkg: unable to activate %s"
msgstr "pkg: невозможно активировать %s"

#: ../modules/client/bootenv.py:274
#, python-format
msgid "pkg: unable to unmount %s"
msgstr "pkg: невозможно демонтировать %s"

#: ../modules/client/bootenv.py:280
#, python-format
msgid ""
"\n"
"A clone of %s exists and has been updated and activated.\n"
"On the next boot the Boot Environment %s will be mounted on '/'.\n"
"Reboot when ready to switch to this updated BE.\n"
msgstr ""
"\n"
"Клон %s существует, он был обновлен и активирован.\n"
"Во время следующей загрузки загрузочная среда %s будет смонтирована в '/'.\n"
"Выполните перезагрузку, когда будете готовы переключиться на обновленную загрузочную среду.\n"

#: ../modules/client/bootenv.py:294
#, python-format
msgid "%s has been updated successfully"
msgstr "Обновление %s выполнено успешно"

#: ../modules/client/bootenv.py:326
#, python-format
msgid ""
" The running system has not been modified. Modifications were only made to a "
"clone of the running system.  This clone is mounted at %s should you wish to "
"inspect it."
msgstr ""
" Работающая система не изменена. Изменения были внесены "
"только в копию работающей системы. Данная копия смонтирована по "
"адресу %s, если необходимо ее проверить."

#: ../modules/client/bootenv.py:336
#, python-format
msgid "pkg: unable to rollback BE %s and restore image"
msgstr "pkg: невозможно выполнить откат загрузочной среды %s и восстановить образ"

#: ../modules/client/bootenv.py:345
#, python-format
msgid "%s failed to be updated. No changes have been made to %s."
msgstr "%s обновить не удалось. В %s не внесено никаких изменений."

#: ../modules/client/bootenv.py:357
#, python-format
msgid "pkg: unable to destroy snapshot %s"
msgstr "pkg: невозможно уничтожить снимок %s"

#: ../modules/client/bootenv.py:386
#, python-format
msgid "pkg: unable to create BE %s"
msgstr "pkg: невозможно создать загрузочную среду %s"

#: ../modules/client/bootenv.py:391
#, python-format
msgid "pkg: unable to mount BE %s on %s"
msgstr "pkg: невозможно смонтировать загрузочную среду %s на %s"

#: ../modules/client/bootenv.py:395
#, python-format
msgid ""
"The Boot Environment %s failed to be updated. A snapshot was taken before "
"the failed attempt and is mounted here %s. Use 'beadm unmount %s' and then "
"'beadm activate %s' if you wish to boot to this BE."
msgstr ""
"Сбой обновления загрузочной среды %s. До сбойной попытки был сделан "
"снимок, он монтирован по адресу %s. Если необходимо выполнить загрузку "
"из данной загрузочной среды, выполните команду 'beadm unmount %s', а затем 'beadm activate %s'."

#: ../modules/client/bootenv.py:408
#, python-format
msgid ""
"The Boot Environment %s failed to be updated. A snapshot was taken before "
"the failed attempt and has been restored so no changes have been made to %s."
msgstr ""
"Сбой обновления загрузочной среды %s. До сбойной попытки был сделан "
"и сохранен снимок, поэтому в %s никаких изменений не внесено."

#: ../modules/client/image.py:1382
msgid "This server is not a valid package depot."
msgstr "Данный сервер не является допустимым хранилищем пакетов."

#: ../modules/client/image.py:2545 ../modules/client/image.py:2620
msgid "Before evaluation:"
msgstr "До оценки:"

#: ../modules/client/image.py:2566 ../modules/client/image.py:2629
msgid "After evaluation:"
msgstr "После оценки:"

#: ../modules/client/image.py:2685
msgid "SUNWipkg update check failed."
msgstr "Сбой проверки обновления SUNWipkg."

#: ../modules/misc.py:442
msgid "B"
msgstr "Б"

#: ../modules/misc.py:443
msgid "kB"
msgstr "кБ"

#: ../modules/misc.py:444
msgid "MB"
msgstr "МБ"

#: ../modules/misc.py:445
msgid "GB"
msgstr "ГБ"

#: ../modules/misc.py:446
msgid "TB"
msgstr "ТБ"

#: ../modules/misc.py:447
msgid "PB"
msgstr "ПБ"

#: ../modules/misc.py:448 ../modules/misc.py:452
msgid "EB"
msgstr "ЭБ"

#: ../modules/server/catalog.py:106
#, python-format
msgid ""
"Index corrupted or out of date. Removing old index directory (%s)  and "
"rebuilding search indexes."
msgstr ""
"Индекс поврежден или устарел. Удаляется старый каталог индекса (%s) и "
"выполняется перестроение поисковых индексов."

#: ../modules/server/catalog.py:122
msgid "writable root not writable by current user id or group."
msgstr "корень не доступен для записи текущему пользователю или группе."

#: ../modules/server/catalog.py:127
msgid "unable to write to index directory."
msgstr "запись в каталог индекса невозможна."

#: ../packagemanager.py:253
msgid "_Current Repository (Name and Description)"
msgstr "_Текущее хранилище (название и описание)"

#: ../packagemanager.py:254
msgid "Search Current Repository (Name and Description)"
msgstr "Поиск по текущему хранилищу (название и описание)"

#: ../packagemanager.py:257
msgid "_All Repositories (Exact Match)"
msgstr "_Все хранилища (точное совпадение)"

#: ../packagemanager.py:258
msgid "Search All Repositories (Exact Match)"
msgstr "Поиск по всем хранилищам (точное совпадение)"

#: ../packagemanager.py:371
msgid ""
"Checking SUNWipkg and SUNWipkg-gui versions\n"
"\n"
"Please wait ..."
msgstr ""
"Проверка версий SUNWipkg и SUNWipkg-gui \n"
"\n"
"Подождите..."

#: ../packagemanager.py:533
#, python-format
msgid "GUI will not respond to any event! %s.Check declare_signals()"
msgstr "GUI не отвечает на события! %s.Проверьте declare_signals()"

#: ../packagemanager.py:603
msgid "All Repositories Search Results"
msgstr "Результаты поиска по всем хранилищам"

#: ../packagemanager.py:632 ../packagemanager.py:2006
#: ../packagemanager.py:2141 ../packagemanager.py:3257
msgid "All"
msgstr "Все"

#: ../packagemanager.py:660
msgid "<html><head></head><body></body></html>"
msgstr "<html><head></head><body></body></html>"

#: ../packagemanager.py:724
#, python-format
msgid ""
"<html><head></head><body><H2>Welcome toPackageManager!</H2><br><font "
"color='#0000FF'>Warning: Unable to load Start Page:<br>%s</font></body></"
"html>"
msgstr ""
"<html><head></head><body><H2>Добро пожаловать в диспетчер пакетов!</H2><br><font "
"color='#0000FF'>Предупреждение: невозможно загрузить начальную страницу:<br>%s</font></body></"
"html>"

#: ../packagemanager.py:768
#, python-format
msgid "%s"
msgstr "%s"

#: ../packagemanager.py:820
msgid "Loading... "
msgstr "Загрузка... "

#: ../packagemanager.py:826
msgid "Stopped"
msgstr "Остановлено"

#: ../packagemanager.py:840
msgid "Done"
msgstr "Готово"

#: ../packagemanager.py:847
#, python-format
msgid ""
"<html><head></head><body><font color='#000000'>                    <a "
"href='stub'></a></font>                    <a href='pm?%s=internal&uri=%"
"s'>                    <IMG SRC = 'startpage_star.png'                     "
"style='border-style: none'></a> <br><br>                    <h2><font "
"color='#0000FF'>Warning: Unable to                     load URL</font></"
"h2><br>%s</body></html>"
msgstr ""
"<html><head></head><body><font color='#000000'>                    <a "
"href='stub'></a></font>                    <a href='pm?%s=internal&uri=%"
"s'>                    <IMG SRC = 'startpage_star.png'                     "
"style='border-style: none'></a> <br><br>                    <h2><font "
"color='#0000FF'>Предупреждение: невозможно                     загрузить адрес URL</font></"
"h2><br>%s</body></html>"

#: ../packagemanager.py:876 ../packagemanager.py:888
msgid "No URI specified"
msgstr "Идентификатор URI не указан"

#: ../packagemanager.py:910
msgid "Empty Action not supported"
msgstr "Пустые действия не поддерживаются"

#: ../packagemanager.py:914
#, python-format
msgid "Action not supported: %s"
msgstr "Действие не поддерживается: %s"

#: ../packagemanager.py:999 ../packagemanager.py:1097 ../updatemanager.py:614
msgid "Name"
msgstr "Имя"

#: ../packagemanager.py:1009
msgid "Status"
msgstr "Статус"

#: ../packagemanager.py:1018
msgid "Repository"
msgstr "Хранилище"

#: ../packagemanager.py:1023
msgid "Description"
msgstr "Описание"

#: ../packagemanager.py:1184 ../packagemanager.py:1254
msgid "Updates Available"
msgstr "Доступны обновления"

#: ../packagemanager.py:1252
msgid "Not Installed"
msgstr "Не установлено"

#. We enable only first section and later we might enable the rest,
#. depending if there are some packages connected with them
#: ../packagemanager.py:1343
msgid "All Categories"
msgstr "Все категории"

#: ../packagemanager.py:1346
msgid "Meta Packages"
msgstr "Метапакеты"

#: ../packagemanager.py:1347
msgid "Applications"
msgstr "Приложения"

#: ../packagemanager.py:1348
msgid "Desktop (GNOME)"
msgstr "Рабочий стол (GNOME)"

#: ../packagemanager.py:1349
msgid "Development"
msgstr "Разработка"

#: ../packagemanager.py:1350
msgid "Distributions"
msgstr "Дистрибутивы"

#: ../packagemanager.py:1351
msgid "Drivers"
msgstr "Драйверы"

#: ../packagemanager.py:1353
msgid "Web Services"
msgstr "Веб-службы"

#: ../packagemanager.py:1356
msgid "All Packages"
msgstr "Все пакеты"

#: ../packagemanager.py:1358
msgid "Installed Packages"
msgstr "Установленные пакеты"

#: ../packagemanager.py:1362
msgid "Non-installed Packages"
msgstr "Неустановленные пакеты"

#: ../packagemanager.py:1365
msgid "Selected Packages"
msgstr "Выбранные пакеты"

#. This can happen if load_catalogs has not been run
#: ../packagemanager.py:1560
msgid ""
"Unable to get status for search results.\n"
"The catalogs have not been loaded.\n"
"Please try after few seconds.\n"
msgstr ""
"Невозможно получить статус для результатов поиска.\n"
"Каталоги не загружены.\n"
"Повторите попытку через несколько секунд.\n"

#: ../packagemanager.py:1667
msgid "Fetching legal information..."
msgstr "Получение правовой информации..."

#. Checking for Add... is fine enough, as the repository
#. name cannot contain "..." in the name.
#: ../packagemanager.py:2053 ../packagemanager.py:2073
#: ../packagemanager.py:2173 ../packagemanager.py:2355
msgid "Add..."
msgstr "Добавить..."

#: ../packagemanager.py:2266
msgid "Refreshing catalogs"
msgstr "Обновление каталогов"

#: ../packagemanager.py:2267
msgid "Refreshing catalogs..."
msgstr "Обновление каталогов..."

#: ../packagemanager.py:2423
msgid "Selected for Removal:"
msgstr "Выбрано для удаления:"

#: ../packagemanager.py:2431
msgid "Selected for Install/Update:"
msgstr "Выбрано для установки/обновления:"

#: ../packagemanager.py:2435 ../packagemanager.py:2440
msgid "Select packages by marking checkbox\n"
msgstr "Выберите пакеты путем установки флажков\n"

#: ../packagemanager.py:2488
msgid "Package Name"
msgstr "Название пакета"

#: ../packagemanager.py:2507
msgid "Fetching description..."
msgstr "Получение описания..."

#: ../packagemanager.py:2511
msgid "Fetching information..."
msgstr "Получение информации..."

#: ../packagemanager.py:2553
msgid ""
"\n"
"This might be caused by network problem while accessing the repository."
msgstr ""
"\n"
"Это может быть связано с проблемами сетевых соединений при доступе к хранилищу."

#: ../packagemanager.py:2556
msgid "Description not available for this package..."
msgstr "Нет описания для данного пакета..."

#: ../packagemanager.py:2559
msgid "Files Details not available for this package..."
msgstr "Нет подробных сведений о файлах для данного пакета..."

#: ../packagemanager.py:2562
msgid "Dependencies info not available for this package..."
msgstr "Нет информации о зависимостях для данного пакета..."

#: ../packagemanager.py:2565
msgid "Information not available for this package..."
msgstr "Нет информации для данного пакета..."

#: ../packagemanager.py:2583
#, python-format
msgid "Root: %s\n"
msgstr "Корень: %s\n"

#: ../packagemanager.py:2584
msgid "Dependencies:\n"
msgstr "Зависимости:\n"

#: ../packagemanager.py:2599
msgid "Summary:\t\t"
msgstr "Сводка:\t\t"

#: ../packagemanager.py:2600
msgid "Size:\t\t\t"
msgstr "Размер:\t\t\t"

#: ../packagemanager.py:2601
msgid "Category:\t\t"
msgstr "Категория:\t\t"

#: ../packagemanager.py:2602
msgid "Installed Version:\t"
msgstr "Установленная версия:\t"

#: ../packagemanager.py:2603
msgid "Latest Version:\t"
msgstr "Последняя версия:\t"

#: ../packagemanager.py:2604
msgid "Packaging Date:\t"
msgstr "Дата упаковки:\t"

#: ../packagemanager.py:2605
msgid "FMRI:\t\t\t"
msgstr "FMRI:\t\t\t"

#: ../packagemanager.py:2606
msgid "Repository:\t\t"
msgstr "Хранилище:\t\t"

#: ../packagemanager.py:2611 ../packagemanager.py:2619 ../updatemanager.py:798
#: ../updatemanager.py:812
msgid "None"
msgstr "Нет"

#: ../packagemanager.py:2647
msgid "Not available"
msgstr "Недоступен"

#: ../packagemanager.py:3031
msgid ""
"Network problem.\n"
"\n"
msgstr ""
"Проблемы с подключеним к сети.\n"
"\n"

#: ../packagemanager.py:3032
msgid "Details:\n"
msgstr "Подробная информация:\n"

#: ../packagemanager.py:3034
msgid " catalogs successfully updated:\n"
msgstr " каталогов успешно обновлено:\n"

#: ../packagemanager.py:3057
msgid "Unknown error"
msgstr "Неизвестная ишибка"

#. Can't happen when all_known is true and no args,
#. but here for completeness.
#: ../packagemanager.py:3100
msgid ""
"Selected repository does not contain any packages.\n"
"Please reload the list of package."
msgstr ""
"В выбранном хранилище отсутствуют пакеты.\n"
"Повторите загрузку списка пакетов."

#: ../packagemanager.py:3342
#, python-format
msgid "Processing package entries: %d of %d"
msgstr "Обработка записей пакетов: %d из %d"

#: ../packagemanager.py:3445
msgid "Loading Repository Information"
msgstr "Загрузка информации о хранилище"

#: ../packagemanager.py:3447
msgid "Fetching package entries ..."
msgstr "Получение записей пакетов..."

#: ../packagemanager.py:3562
msgid "Searching..."
msgstr "Поиск..."

#: ../packagemanager.py:3573
#, python-format
msgid "%s: first %d found"
msgstr "%s: найдены первые %d"

#: ../packagemanager.py:3575
#, python-format
msgid "%s: %d found"
msgstr "%s: найдено %d "

#: ../packagemanager.py:3596
#, python-format
msgid "%d listed"
msgstr "%d в списке"

#: ../packagemanager.py:3597
#, python-format
msgid "%d selected"
msgstr "Выбрано: %d"

#: ../packagemanager.py:3598
#, python-format
msgid "%d installed"
msgstr "Установлено: %d"

#: ../packagemanager.py:3599
#, python-format
msgid "%s: %s , %s, %s."
msgstr "%s: %s , %s, %s."

#: ../packagemanager.py:3694
msgid ""
"The Update All action is now complete and Package Manager will close.\n"
"\n"
"Review the posted release notes before rebooting your system:\n"
"\n"
msgstr ""
"Обновление всех пакетов закончено, и диспетчер пакетов закрывается.\n"
"\n"
"Прочтите соответствующие замечания о выпуске перед перезагрузкой системы:\n"
"\n"

#: ../packagemanager.py:3724
msgid "Save selected..."
msgstr "Сохранить выбранные..."

#: ../packagemanager.py:3725
msgid "Save selected packages..."
msgstr "Сохранить выбранные пакеты..."

#: ../packagemanager.py:3726
msgid ""
"The installed package(s) require a reboot before installation can be "
"completed."
msgstr ""
"Для завершения установки пакета (пакетов) требуется "
"перезагрузка."

#: ../publish.py:78
msgid ""
"Usage:\n"
"        pkgsend [options] command [cmd_options] [operands]\n"
"\n"
"Packager subcommands:\n"
"        pkgsend create-repository\n"
"        pkgsend open [-en] pkg_fmri\n"
"        pkgsend add action arguments\n"
"        pkgsend import [-T file_pattern] bundlefile ...\n"
"        pkgsend include [-d basedir] manifest ...\n"
"        pkgsend close [-A]\n"
"\n"
"        pkgsend rename src_fmri dest_fmri\n"
"\n"
"Options:\n"
"        -s repo_uri     target repository URI\n"
"        --help or -?    display usage message\n"
"\n"
"Environment:\n"
"        PKG_REPO"
msgstr ""
"Использование:\n"
"        pkgsend [параметры] команда [параметры команды] [операнды]\n"
"\n"
"Субкоманды упаковщика:\n"
"        pkgsend create-repository\n"
"        pkgsend open [-en] pkg_fmri\n"
"        pkgsend add action arguments\n"
"        pkgsend import [-T file_pattern] bundlefile ...\n"
"        pkgsend include [-d basedir] manifest ...\n"
"        pkgsend close [-A]\n"
"\n"
"        pkgsend rename src_fmri dest_fmri\n"
"\n"
"Параметры:\n"
"        -s repo_uri     идентификатор URI целевого хранилища\n"
"        --help or -?    вывод сообщения об использовании\n"
"\n"
"Среда:\n"
"        PKG_REPO"

#: ../publish.py:104
msgid "command does not take operands"
msgstr "команда не принимает операнды"

#: ../publish.py:128
msgid "only -e or -n may be specified"
msgstr "допускаются только параметры -e или -n."

#: ../publish.py:131
msgid "open requires one package name"
msgstr "для команды открытия требуется одно имя пакета"

#: ../publish.py:157
msgid "No transaction ID specified using -t or in $PKG_TRANS_ID."
msgstr "Не указан идентификатор транзакции параметром -t или переменной $PKG_TRANS_ID."

#: ../publish.py:171 ../publish.py:219 ../publish.py:271
msgid "No transaction ID specified in $PKG_TRANS_ID"
msgstr "В $PKG_TRANS_ID не указан идентификатор транзакции"

#: ../publish.py:175 ../publish.py:201 ../publish.py:223 ../publish.py:283
msgid "No arguments specified for subcommand."
msgstr "Не указаны аргументы субкоманды."

#: ../publish.py:181
msgid "A filename must be provided for this action."
msgstr "Для этого действия требуется имя файла."

#: ../publish.py:323
#, python-format
msgid "pkgsend: illegal global option -- %s"
msgstr "pkgsend: недопустимая глобальная опция -- %s"

#: ../pull.py:51
msgid ""
"Usage:\n"
"        pkgrecv -s server [-d dir] pkgfmri ...\n"
"        pkgrecv -s server -n"
msgstr ""
"Использование:\n"
"        pkgrecv -s server [-d dir] pkgfmri ...\n"
"        pkgrecv -s server -n"

#: ../pull.py:75
#, python-format
msgid "Unable to open manifest: %s"
msgstr "Невозможно открыть манифест: %s"

#: ../pull.py:111
#, python-format
msgid "Unable to download files from: %s"
msgstr "Невозможно загрузить файлы из: %s"

#: ../pull.py:120 ../pull.py:165
#, python-format
msgid "Unable to create directory: %s"
msgstr "Невозможно создать каталог: %s"

#: ../pull.py:138
#, python-format
msgid "Unable to extract file: %s"
msgstr "Невозможно извлечь файл: %s"

#: ../pull.py:153
#, python-format
msgid "Unable to download manifest %s from %s"
msgstr "Невозможно загрузить манифест %s из %s"

#: ../pull.py:173
#, python-format
msgid "Unable to open file: %s"
msgstr "Невозможно открыть файл: %s"

#: ../pull.py:180
#, python-format
msgid "Error occurred while reading from: %s"
msgstr "Произошла ошибка во время чтения из: %s"

#: ../pull.py:186
#, python-format
msgid "Error occurred while writing to: %s"
msgstr "Произошла ошибка во время записи в: %s"

#: ../pull.py:225
#, python-format
msgid "Unable to download catalog from: %s"
msgstr "Невозможно загрузить каталог из: %s"

#: ../pull.py:235
#, python-format
msgid "Error while reading from: %s"
msgstr "Ошибка во время чтения из: %s"

#: ../pull.py:262
#, python-format
msgid "Illegal option -- %s"
msgstr "Недопустимая опция -- %s"

#: ../pull.py:275
msgid "must specify a server"
msgstr "необходимо указать сервер"

#: ../pull.py:282
msgid "-n takes no options"
msgstr "параметр -n используется без опций"

#: ../pull.py:290
msgid "must specify at least one pkgfmri"
msgstr "необходимо указать хотя бы один pkgfmri"

#: ../pull.py:302
#, python-format
msgid "%(fmri)s is an illegal fmri: %(error)s"
msgstr "%(fmri)s не является действительным fmri: %(error)s"

#: ../pull.py:315
msgid "No files to retrieve."
msgstr "Нет файлов для извлечения."

#: ../um/data/updatemanager-preferences.schemas.in.h:1
msgid ""
"A text message describing the updates is displayed near the panel's "
"Notification Area when updates become available if set to true."
msgstr ""
"Текстовое сообщение с описанием состояния обновлений отображается "
"возле области уведомлений панели в случае наличия доступных обновлений, если параметр имеет значение \"истина\"."

#: ../um/data/updatemanager-preferences.schemas.in.h:2
msgid "How often to check for updates"
msgstr "Как часто проверять наличие обновлений"

#: ../um/data/updatemanager-preferences.schemas.in.h:3
msgid ""
"How often to check for updates. The valid values are Daily, Weekly, Monthly, "
"Never"
msgstr ""
"Как часто проверять наличие обновлений. допустимые значения: ежедневно, "
"еженедельно, ежемесячно, никогда"

#: ../um/data/updatemanager-preferences.schemas.in.h:4
msgid "Start delay"
msgstr "Задержка выполнения"

#: ../um/data/updatemanager-preferences.schemas.in.h:5
msgid "The notify icon is displayed on startup if set to true."
msgstr "Если параметр имеет значение \"истина\", значок уведомления отображается во время запуска"

#: ../um/data/updatemanager-preferences.schemas.in.h:6
msgid ""
"The time, in seconds, that the updatemanagernotifier should wait, after "
"starting, before checking for updates."
msgstr ""
"Время в секундах, которое средство уведомления диспетчера обновлений должно выждать "
"после запуска перед проверкой обновлений."

#: ../um/data/updatemanager-preferences.schemas.in.h:7
msgid ""
"The updatemanagernotifier program terminates after the user activates the "
"icon in the notification area if set to true."
msgstr ""
"Если установлено значение true, программа updatemanagernotifier завершает работу после выбора пользователем значка "
"в области уведомлений."

#: ../um/data/updatemanager-preferences.schemas.in.h:8
msgid "Whether to always display the icon on startup"
msgstr "Всегда ли отображать значок во время запуска"

#: ../um/data/updatemanager-preferences.schemas.in.h:9
msgid "Whether to display text message when updates are available"
msgstr "Отображать ли текстовые сообщения, когда доступны обновления"

#: ../um/data/updatemanager-preferences.schemas.in.h:10
msgid "Whether to terminate updatemanagernotiifer after icon activation."
msgstr "Необходимость прекращения работы updatemanagernotifier после активации значка."

#: ../um/data/updatemanager.desktop.in.h:1
msgid "Install available updates"
msgstr "Установить доступные обновления"

#: ../um/data/updatemanager.desktop.in.h:2 ../um/data/updatemanager.glade.h:10
#: ../updatemanager.py:927 ../updatemanager.py:992
#: ../updatemanagernotifier.py:351
msgid "Update Manager"
msgstr "Диспетчер обновлений"

#: ../um/data/updatemanager.glade.h:1
msgid "<b>Evaluate</b>"
msgstr "<b>Оценка</b>"

#: ../um/data/updatemanager.glade.h:2
msgid "<b>Latest Version _Details</b>"
msgstr "<b>_Подробные данные о последней версии</b>"

#: ../um/data/updatemanager.glade.h:3
msgid "<b>_Details</b>"
msgstr "<b>_Подробные сведения</b>"

#: ../um/data/updatemanager.glade.h:5
msgid "Close dialog when _finished"
msgstr "Закрыть диалог после _окончания"

#: ../um/data/updatemanager.glade.h:7 ../updatemanager.py:207
#: ../updatemanager.py:429 ../updatemanager.py:436 ../updatemanager.py:443
#: ../updatemanager.py:450 ../updatemanager.py:457
msgid "Index"
msgstr "Индекс"

#: ../um/data/updatemanager.glade.h:8
msgid "Install Update"
msgstr "Установить обновление"

#: ../um/data/updatemanager.glade.h:11 ../updatemanager.py:594
msgid ""
"Updates are available for the following packages.\n"
"Select the packages you want to update and click Install."
msgstr ""
"Для данных пакетов доступны обновления.\n"
"Выберите пакеты для обновления и нажмите кнопку \"Установить\"."

#: ../um/data/updatemanager.glade.h:13
msgid "_Cancel"
msgstr "О_тменить"

#: ../um/data/updatemanager.glade.h:14
msgid "_Install Updates"
msgstr "_Установить обновления"

#: ../um/data/updatemanager.glade.h:15
msgid "_OK"
msgstr "_OK"

#: ../um/data/updatemanager.glade.h:16
msgid "_Select all Updates"
msgstr "_Выбрать все обновления"

#: ../um/data/updatemanagernotifier.desktop.in.h:1
msgid "Notify user when updates are available"
msgstr "Уведомлять пользователя о наличии обновлений"

#: ../um/data/updatemanagernotifier.desktop.in.h:2
msgid "Update Manager Notifier"
msgstr "Средство уведомления диспетчера обновлений"

#: ../updatemanager.py:123
#, python-format
msgid "Fetching catalog: '%s' ..."
msgstr "Получение каталога: '%s' ..."

#: ../updatemanager.py:127
msgid "Fetching catalog"
msgstr "Получение каталога"

#: ../updatemanager.py:130
msgid "Creating Plan ... "
msgstr "Создание плана... "

#: ../updatemanager.py:141
msgid "Creating Plan"
msgstr "Создание плана"

#: ../updatemanager.py:152
#, python-format
msgid "Verifying: %s ..."
msgstr "Проверка: %s ..."

#: ../updatemanager.py:159
msgid "Verifying"
msgstr "Проверка"

#: ../updatemanager.py:173
msgid "Completed"
msgstr "Завершено"

#: ../updatemanager.py:387
#, python-format
msgid "GUI will not respond to any event! %s. Check updatemanager.py signals"
msgstr "Графический интерфейс не реагирует на события! %s. Проверьте сигналы updatemanager.py"

#: ../updatemanager.py:426 ../updatemanager.py:433 ../updatemanager.py:440
#: ../updatemanager.py:447 ../updatemanager.py:454
msgid "Evaluate"
msgstr "Оценка"

#: ../updatemanager.py:462
msgid "Evaluate - canceling..."
msgstr "Оценка - отмена..."

#: ../updatemanager.py:466
msgid "Download - canceling..."
msgstr "Загрузка - отмена"

#: ../updatemanager.py:471
msgid "Evaluate - failed"
msgstr "Оценка - сбой"

#: ../updatemanager.py:475
msgid "Download - failed"
msgstr "Загрузка - сбой"

#: ../updatemanager.py:479
msgid "Install - failed"
msgstr "Установка - сбой"

#: ../updatemanager.py:483
msgid "Index - failed"
msgstr "Индекс - сбой"

#: ../updatemanager.py:601
msgid ""
"Updates are available for the following packages.\n"
"Click Update All to create a new boot environment and install all packages "
"into it."
msgstr ""
"Обновления доступны для следующих пакетов.\n"
"Нажмите \"Обновить все\" для создания новой загрузочной среды "
"и установки в нее всех пакетов."

#: ../updatemanager.py:629
msgid "Latest Version"
msgstr "Последняя версия"

#: ../updatemanager.py:638
msgid "Size (Meg)"
msgstr "Размер (мегабайты)"

#: ../updatemanager.py:717
#, python-format
msgid "count: %d"
msgstr "счетчик: %d"

#: ../updatemanager.py:742 ../updatemanager.py:752
msgid "Error\n"
msgstr "Ошибка\n"

#: ../updatemanager.py:745
#, python-format
msgid "'%s' is not an install image\n"
msgstr "'%s' не является образом установки\n"

#: ../updatemanager.py:755
#, python-format
msgid "Version mismatch: expected %s received %s\n"
msgstr "Несоответствие версий: ожидалась %s, а получена %s\n"

#: ../updatemanager.py:760
msgid "<b>No Updates available.</b>"
msgstr "<b>Обновления отсутствуют.</b>"

#: ../updatemanager.py:817
#, python-format
msgid ""
"\n"
"Summary:\t\t\t%s\n"
"Size:       \t\t\t%s\n"
"Category:\t\t\t%s\n"
"Latest Version:\t\t%s\n"
"Installed Version:\t%s\n"
"Packaging Date:\t%s\n"
"FMRI:       \t\t\t%s\n"
"Repository:       \t\t%s\n"
msgstr ""
"\n"
"Сводка:\t\t\t%s\n"
"Размер:       \t\t\t%s\n"
"Категория:\t\t\t%s\n"
"Последняя версия:\t\t%s\n"
"Установленная версия:\t%s\n"
"Дата упаковки:\t%s\n"
"FMRI:       \t\t\t%s\n"
"Хранилище:       \t\t%s\n"

#: ../updatemanager.py:847 ../updatemanagernotifier.py:310
#, python-format
msgid "%s is not valid image, trying root image"
msgstr "%s - недопустимый образ, попытка использования корневого образа"

#: ../updatemanager.py:857 ../updatemanagernotifier.py:320
#, python-format
msgid "%s is not valid root image, return None"
msgstr "%s - недопустимый корневой образ, возврат \"Нет\""

#: ../updatemanager.py:896
#, python-format
msgid ""
"\n"
"Fetching details for %s ..."
msgstr ""
"\n"
"Получение подробных сведений для %s ..."

#: ../updatemanager.py:920
msgid ""
"\n"
"No details available"
msgstr ""
"\n"
"Нет доступных подробных сведений"

#: ../updatemanager.py:971
msgid ""
"\n"
"Canceling update, please wait ..."
msgstr ""
"\n"
"Подождите, отмена обновления  ..."

#: ../updatemanager.py:976
msgid ""
"\n"
"Unable to cancel at this time."
msgstr ""
"\n"
"Невозможно отменить сейчас."

#: ../updatemanager.py:979
msgid "Installing Updates"
msgstr "Установка обновлений"

#: ../updatemanager.py:1002
msgid "ERROR"
msgstr "ОШИБКА"

#: ../updatemanager.py:1004
msgid "Following Incorporated package(s) cannot be updated:"
msgstr "Невозможно обновить следующие объединенные пакеты:"

#: ../updatemanager.py:1008
msgid "Update using: Update All\n"
msgstr "Используется обновление: Обновить все\n"

#: ../updatemanager.py:1025 ../updatemanager.py:1155 ../updatemanager.py:1190
msgid ""
"\n"
"ERROR"
msgstr ""
"\n"
"ОШИБКА"

#: ../updatemanager.py:1028
#, python-format
msgid ""
"%s\n"
"%s"
msgstr ""
"%s\n"
"%s"

#: ../updatemanager.py:1040
#, python-format
msgid ""
"\n"
"Update All finished successfully in %1.f mins\n"
"\n"
msgstr ""
"\n"
"Действие \"Обновить все\" завершено успешно в течение %1.f мин\n"
"\n"

#: ../updatemanager.py:1045
msgid ""
"\n"
"Update All finished successfully in < 1 min\n"
"\n"
msgstr ""
"\n"
"Действие \"Обновить все\" завершено успешно в течение < 1 мин\n"
"\n"

#: ../updatemanager.py:1049
msgid ""
"Review the posted release notes before rebooting your system:\n"
"\n"
msgstr ""
"Прочтите соответствующие замечания о выпуске перед перезагрузкой системы:\n"
"\n"

#: ../updatemanager.py:1059
msgid "Update All Completed"
msgstr "Команда \"Обновить все\" выполнена"

#: ../updatemanager.py:1074
msgid "Nothing selected to update."
msgstr "Для обновления ничего не выбрано."

#: ../updatemanager.py:1081
msgid "Updating ..."
msgstr "Обновление..."

#: ../updatemanager.py:1087
msgid ""
"\n"
"Evaluate\n"
msgstr ""
"\n"
"Оценка\n"

#: ../updatemanager.py:1093
msgid "Update error in plan install:"
msgstr "Ошибка обновления во время установки плана:"

#: ../updatemanager.py:1103
msgid "Update unexpected API error:"
msgstr "Неожиданная ошибка API обновления:"

#: ../updatemanager.py:1108
msgid "Update unexpected error:"
msgstr "Неожиданная ошибка обновления:"

#: ../updatemanager.py:1131
msgid "Packages to be installed:"
msgstr "Пакеты, которые необходимо установить:"

#: ../updatemanager.py:1135
msgid "Update"
msgstr "Обновить"

#: ../updatemanager.py:1136
msgid "Update finished successfully."
msgstr "Обновление успешно завершено."

#: ../updatemanager.py:1157
#, python-format
msgid ""
"%s Download failed:\n"
"%s"
msgstr ""
"%s Сбой загрузки:\n"
"%s"

#: ../updatemanager.py:1164 ../updatemanager.py:1199
#, python-format
msgid "%s exceded available disc space"
msgstr "%s превысил доступное место на диске"

#: ../updatemanager.py:1169 ../updatemanager.py:1174 ../updatemanager.py:1204
#: ../updatemanager.py:1209
#, python-format
msgid "%s unexpected error:"
msgstr "%s -- неожиданная ошибка:"

#: ../updatemanager.py:1192
#, python-format
msgid ""
"%s Execute plan failed:\n"
"%s"
msgstr ""
"%s -- сбой выполнения плана:\n"
"%s"

#: ../updatemanager.py:1226
msgid ""
"Not enough disc space, the Update All action cannot be performed.\n"
"\n"
"Click OK to manage your existing BEs and free up disk space or Cancel to "
"cancel Update All."
msgstr ""
"Недостаточно места на диске, выполнение обновления всех пакетов невозможно.\n"
"\n"
"Нажмите кнопку OK для выполнения управления имеющимися средами загрузки для освобождения дискового пространства либо нажмите кнопку Cancel для отмены"
"обновления всех пакетов."

#: ../updatemanager.py:1230
msgid "Not Enough Disc Space"
msgstr "Недостаточно места на диске"

#: ../updatemanager.py:1398
msgid ""
"\n"
"Download\n"
msgstr ""
"\n"
"Загрузить\n"

#: ../updatemanager.py:1405
#, python-format
msgid "\tpkg %d/%d: \tfiles %d/%d \txfer %.2f/%.2f(meg)"
msgstr "\tpkg %d/%d: \tfiles %d/%d \txfer %.2f/%.2f(МБ)"

#: ../updatemanager.py:1422
msgid ""
"\n"
"Install\n"
msgstr ""
"\n"
"Установить\n"

#: ../updatemanager.py:1424 ../updatemanager.py:1428
#, python-format
msgid "\t%s\t%d/%d actions"
msgstr "\t%s\t%d/%d действия"

#: ../updatemanager.py:1446
msgid "Index\n"
msgstr "Индекс\n"

#: ../updatemanager.py:1447 ../updatemanager.py:1451
#, python-format
msgid "\t%-25s\t%d/%d actions"
msgstr "\t%-25s\t%d/%d действия"

#: ../updatemanager.py:1533
msgid "Checking for new software"
msgstr "Проверка наличия нового программного обеспечения"

#: ../updatemanagernotifier.py:330
msgid "Updates are available"
msgstr "Имеются доступные обновления"

#: ../updatemanagernotifier.py:352
msgid ""
"Updates available\n"
"Please click on icon to update."
msgstr ""
"Доступные обновления\n"
"Нажмите значок для обновления."

#: ../updatemanagernotifier.py:427
msgid "Another instance of UpdateManagerNotify is running"
msgstr "Запущен еще один экземпляр средства уведомления диспетчера обновлений"

# ../gui/data/packagemanager-info.xml.in:31
msgid "pkg(5) info file"
msgstr "Файл info для pkg(5)"
